---

title: Methods for determining protein binding specificity using peptide libraries
abstract: A method for determining protein binding specificity using a screen of a peptide library is provided. The method can be used to determine binding specificity for human NAD-dependent deacetylase SIRT1, and to identify the most efficiently deacetylated peptide sequences. The method can be also used to screen a combinatorial H4 histone N-terminal tail peptide library to examine the binding preferences of a Î±-phos (S1) H4 antibody toward all known possible H4 histone modification states.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08119572&OS=08119572&RS=08119572
owner: Wisconsin Alumni Research Foundation
number: 08119572
owner_city: Madison
owner_country: US
publication_date: 20061024
---
This invention claims priority to U.S. Provisional Patent Application Ser. No. 60 729 866 filed Oct. 24 2005.

This invention was made with United States government support awarded by the NIH grants Nos. GM065386 and GM059785. The United States government has may have certain rights in this invention.

This invention relates to a method for determining substrate specificity of protein binding using peptide libraries.

An emerging paradigm for discovery in pharmaceutical and related biotechnology is the assembly of novel synthetic compound libraries by new methods of solid phase combinatorial synthesis. Combinatorial chemistry refers to a set of strategies for the parallel synthesis of multiple compounds or compounds mixtures either in solution or on solid supports in the form of polymer based resins beads .

One implementation of combinatorial synthesis that is suitable to produce large chemical libraries relies on one bead one compound OBOC libraries which contain from 10to 10compounds. These libraries are screened by performing a variety of chemical and biochemical assays to identify individual compounds eliciting a response. The chemical identity of such specific compounds can then be determined by direct analysis using e.g. micro sequencing and mass spectrometry.

In peptidic OBOC libraries peptide sequences from a group of selected amino acid building blocks are represented in an on bead format in which many copies of only one sequence exist on each bead. OBOC libraries permit sifting through a list of peptide substrate sequences to correlate top hits with protein sequence databases. This strategy has been used successfully for example to determine the optimal peptide substrates of peptide deformylase a Fe metalloenzyme that catalyzes N terminal deformylation of nascent polypeptides in eubacteria Hu et al. 1999 38 643 650 .

Histone proteins serve to package DNA and to regulate its accessibility for processes including transcription repair and replication. Six major histone classes are known. Two each of the class H2A H2B H3 and H4 assemble to form one nucleosome core particle around which DNA is wrapped. Acting as spools around which DNA winds histones play a role in gene regulation. Histones achieve this control over DNA by acting as substrates to a host of posttranslational modifications that dictate function. Particularly dense with posttranslational information are the N terminal histone tails. These can be covalently modified at several sites. Modifications of the histone tail include methylation acetylation phosphorylation ubiquitination sumoylation citrullination and ADP ribosylation. The core of the histones can also be modified. Combinations of modifications are thought to constitute a code the so called histone code . The histone code hypothesis asserts that histone binding proteins and histone modifying enzymes read and interpret the posttranslational states of properly primed histones to facilitate a particular outcome such as gene silencing transcription mitosis etc. Strahl et al. 2000 403 41 45 . However thus far the combinatorial complexity of the histone modification patterns has precluded a systematic inquiry of the patterns recognized by these code readers proteins and enzymes that would display preferential specificity for these context dependent modifications.

Protein deacetylases have been implicated in a variety of disease states including aging diabetes HIV regulation cancer cardiovascular disorders and neurodegenerative diseases. Histone deacetylase inhibitors are currently in clinical trials as cancer treatments. In particular the Silent information regulator 2 Sir2 family of NAD dependent protein deacetylases has been studied in recent years. This burgeoning interest can be attributed to the important roles of Sir2 enzymes sirtuins in regulating chromatin architecture promoting transcriptional silencing and longevity and in fatty acid metabolism. NAD dependent lysyl deacetylation is characterized by the stoichiometric release of nicotinamide and a novel metabolite O acetyl ADP ribose OAADPr .

The Sir2 family of deacetylases is highly conserved among all forms of life with seven known human homologs SIRT1 7 . The most studied mammalian homolog SIRT1 is a nuclear enzyme that has been found to deacetylate a number of proteins. Histones H3 and H4 p53 p300 TAF68 PCAF MyoD PGC 1alpha FOXO1 and 4 NF kappaB and Tat are examples reported to be either biological targets and or in vitro substrates of SIRT1. Collectively the variety of proposed physiological targets reflects the functional diversity of SIRT1.

Identifying biological substrates is an important step in understanding the molecular basis for sirtuin phenotypes. However in many of the studied cases a certain degree of logical bias was used to link the target protein and SIRT1 as unbiased global substrate screening procedures were not used. Varying conclusions have been reached in regard to sirtuin substrate specificity and recognition. Most striking are the conclusions that sirtuins display minimal side chain recognition Avalos et al. 2002 10 523 535 Zhao et al. 2003 11 1403 1411 and that SIRT1 displays no substrate sequence specificity Blander et al. 2005 280 9780 9785 . In contrasting reports clear substrate preferences were noted for yeast Sir2 and HST2 Borra et al. 2004 43 9877 9887 and human SIRT2 North et al. 2003 11 437 444 .

To date only one study has attempted to probe sirtuin substrate specificity using an acetyl peptide library approach Blander et al. 2005 . Curiously the study reported that SIRT1 displayed no substrate specificity in vitro a conclusion based on an oriented peptide library. With this method only globally preferred amino acids could be resolved and the actual sequence of individual peptides was not elucidated. Although the peptide library technique has been successful for examining protein kinase substrate specificity Songyang et al. 1994 4 973 982 its usefulness toward protein deacetylases remains uncertain.

This invention provides a method for determining binding specificity of a protein deacetylase which includes contacting the protein deacetylase with a peptide library. The peptide library includes a plurality of solid phase supports where each solid phase support is linked to a different and distinct peptide. The method includes labeling the peptide library with a label specific for an amino group formed upon deacetylation and correlating the label intensity with binding specificity of the protein deacetylase.

The label can be specific for an amino group formed upon deacetylation. The solid phase supports used to practice the method can be beads. The label can be colorimetric radioactive or fluorescent. The label can also be a labeled quantum dot.

The method can include sorting the solid phase supports from the labeled peptide library on the basis of label intensity. The method can include determining the sequence of the peptide attached to the solid phase support.

The protein deacetylase assayed by the method can be a sirtuin. Preferably the protein deacetylase is SIRT1.

The peptide sequence can be selected from the group consisting of LNKDQ SEQ ID NO 26 WHKFQ SEQ ID NO 27 WHKFE SEQ ID NO 28 SYKQW SEQ ID NO 29 QPKQI SEQ ID NO 30 VQKII SEQ ID NO 31 HRKMP SEQ ID NO 32 HKKMP SEQ ID NO 33 AVKFM SEQ ID NO 34 NHKLL SEQ ID NO 35 RFKPE SEQ ID NO 36 KFKPE SEQ ID NO 37 FEKYR SEQ ID NO 40 MMKQQ SEQ ID NO 38 WGKSP SEQ ID NO 39 FEKYK SEQ ID NO 41 WPKWQ SEQ ID NO 42 RAKMD SEQ ID NO 43 KAKMD SEQ ID NO 44 GTKTG SEQ ID NO 45 GYKPT SEQ ID NO 46 IFKTF SEQ ID NO 47 TEKQE SEQ ID NO 48 HWKTH SEQ ID NO 49 DSKGA SEQ ID NO 50 SDKYH SEQ ID NO 51 NHKII SEQ ID NO 52 WWKHG SEQ ID NO 53 PIKEQ SEQ ID NO 54 RPKQF SEQ ID NO 55 KPKQF SEQ ID NO 56 DVKMH SEQ ID NO 57 IYKND SEQ ID NO 58 TPKNA SEQ ID NO 59 PGKLY SEQ ID NO 60 RWKIT SEQ ID NO 61 KWKIT SEQ ID NO 62 WRKIT SEQ ID NO 63 WKKIT SEQ ID NO 64 WPKIT SEQ ID NO 65 PWKIT SEQ ID NO 66 RPKSI SEQ ID NO 67 KPKSI SEQ ID NO 68 PRKSI SEQ ID NO 69 and PKKSI SEQ ID NO 70 .

This invention provides a method for determining binding specificity of an enzyme. The method includes contacting a combinatorial peptide library with the enzyme where the peptide library comprises a plurality of solid phase supports where each solid phase support is linked to a different and distinct peptide labeling peptides with a label specific for peptides covalently modified by the enzyme and correlating the intensity of the label with peptides that are covalently modified by the enzyme thereby determining binding specificity of the enzyme. The covalent modification may include a post translational modification.

This invention provides an analytical method which includes generating a combinatorial library of peptides that includes one or more peptide sequences attached to a solid phase support where the combinatorial library includes a plurality of solid phase supports linked to a different and distinct peptide and where each peptide comprises two or more chemically modified amino acids contacting the combinatorial library with a protein detecting the protein bound to one or more peptides using a label and correlating the label intensity with peptides to which the protein binds thereby determining the binding specificity of the protein. The solid phase supports can be beads. The label can be calorimetric radioactive or fluorescent. The label can be a labeled quantum dot. The solid phase supports from the labeled peptide library can be sorted on the basis of label intensity.

The method can include determining the sequence of the peptide attached to the solid phase support. The method can include determining the modification status of the peptide attached to the solid phase support.

Each peptide sequence can include at least 5 amino acids. At least one peptide sequence can include covalently modified amino acid.

The covalent modification can include methylation acetylation phosphorylation ubiquitination sumoylation citrullination or ADP ribosylation. The protein that is used to contact the combinatorial peptide library can be an enzyme. The protein that is used to contact the combinatorial peptide library can be an antibody. The combinatorial library can include one or more N terminal peptide sequences from a histone.

The practice of the present invention employs unless otherwise indicated conventional techniques of peptide synthesis molecular biology including recombinant techniques microbiology cell biology biochemistry immunology protein kinetics and mass spectroscopy which are within the skill of art. Such techniques are explained fully in the literature e.g. in Bodanszky et al. 1976 2ed. John Wiley and Sons Sambrook et al. 20003rd ed. Cold Spring Harbor Laboratory Press Volumes 1 3 John Wiley Sons Inc. Kriegler 1990 . Stockton Press New York Dieffenbach et al. 1995 Cold Spring Harbor Laboratory Press each of which is incorporated herein by reference in its entirety.

Generally the nomenclature and the laboratory procedures in recombinant DNA technology described below are those well known and commonly employed in the art. Standard techniques are used for cloning DNA and RNA isolation amplification and purification. Generally enzymatic reactions involving DNA ligase DNA polymerase restriction endonucleases and the like are performed according to the manufacturer s specifications. Procedures employing commercially available assay kits and reagents are typically used according to manufacturer defined protocols unless otherwise noted.

The term acetyl sometimes called ethanoyl is a functional group the acyl of acetic acid with chemical formula COCH.

A label is a composition detectable by spectroscopic photochemical biochemical immunochemical or chemical means. For example useful labels include P fluorescent dyes colorimetric labels electron dense reagents enzymes e.g. as commonly used in an ELISA biotin digoxigenin or quantum dots. As used herein the term label also includes indirect labeling of proteins using detectable labels bound to other molecules or complexes of molecules that bind to a protein of interest including antibodies and proteins to which antisera or monoclonal antibodies specifically bind. As used herein the term colorimetric label includes a label that is detected using an enzyme linked assay.

 Antibodies as used herein includes polyclonal and monoclonal antibodies chimeric and single chain antibodies as well as Fab fragments including the products of a Fab or other immunoglobulin expression library. With respect to antibodies the term immunologically specific refers to antibodies that bind to one or more epitopes of a protein of interest but which do not substantially recognize and bind other molecules in a sample containing a mixed population of antigenic biological molecules.

The terms isolated purified or biologically pure refer to material that is substantially or essentially free from components that normally accompany it as found in its native state. Purity and homogeneity are typically determined using analytical chemistry techniques such as polyacrylamide gel electrophoresis or high performance liquid chromatography. A peptide or protein that is the predominant species present in a preparation is substantially purified. The term purified denotes that a peptide or protein gives rise to essentially one band in an electrophoretic gel or HPLC spectrum. Particularly it means that the peptide or protein is at least 85 pure more preferably at least 95 pure and most preferably at least 99 pure.

Two peptides or polypeptides are said to be identical if the sequence of amino acid residues in the two sequences is the same when aligned for maximum correspondence as described below.

Peptide or protein sequence identities are evaluated using the Basic Local Alignment Search Tool BLAST which is well known in the art Karlin and Altschul 1990 87 2267 2268 Altschul et al. 1997 25 3389 3402 . The BLAST programs can be used with the default parameters or with modified parameters provided by the user.

 Percentage of sequence identity is determined by comparing two optimally aligned sequences over a comparison window wherein the portion of the peptide or polypeptide sequence in the comparison window may comprise additions or deletions i.e. gaps as compared to the reference sequence which does not comprise additions or deletions for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by 100 to yield the percentage of sequence identity.

The term substantial identity of amino acid sequences for purposes of this invention normally means peptide or polypeptide sequence identity of at least 40 . Preferred percent identity of peptides or polypeptides can be any integer from 40 to 100 . More preferred embodiments include at least 40 45 50 55 60 65 70 75 80 85 86 87 88 89 90 91 92 93 94 95 96 97 98 98.7 or 99 .

Peptides or polypeptides that are substantially similar share sequences as noted above except that residue positions which are not identical may differ by conservative amino acid changes. Conservative amino acid substitutions refer to the interchangeability of residues having similar side chains. For example a group of amino acids having aliphatic side chains is glycine alanine valine leucine and isoleucine a group of amino acids having aliphatic hydroxyl side chains is serine and threonine a group of amino acids having amide containing side chains is asparagine and glutamine a group of amino acids having aromatic side chains is phenylalanine tyrosine and tryptophan a group of amino acids having basic side chains is lysine arginine and histidine and a group of amino acids having sulfur containing side chains is cysteine and methionine. Preferred conservative amino acids substitution groups are valine leucine isoleucine phenylalanine tyrosine lysine arginine alanine valine aspartic acid glutamic acid and asparagine glutamine.

The term combinatorial library refers to a collection of compounds synthesized in parallel or as a collection of compounds synthesized with mixtures of reagents or employing a split and mix methodology from a set of defined building blocks and using these building blocks in many combinations to generate a complex library of novel compounds. One implementation of combinatorial synthesis is the generation of one bead one compound OBOC libraries where each novel compound is represented on a single bead. These libraries can be screened by performing a variety of chemical and biochemical assays to identify individual compounds eliciting a response. The identity of the compound on the support is either known by prior knowledge from direct parallel synthetic procedures or is determined by direct analysis after detection using e.g. micro sequencing and mass spectrometry.

In peptidic OBOC libraries peptide from a group of selected amino acid building blocks can be represented in an on bead format in which many copies of only one sequence and of only one post translationally modified state exist on each bead. Alternatively peptides may be attached to other suitable types of support including microarrays microplates chips or other surfaces that are suitable for detecting protein binding events.

The invention described here uses both modified and unmodified amino acids as building blocks to create combinatorial peptide libraries to evaluate the binding specificities of proteins. These peptide libraries can vary in amino acid sequence and in modification state e.g. methylation acetylation or phosphorylation. The chemical identity amino acid sequence and post translational status of these peptides can be determined by direct analysis using e.g. micro sequencing and mass spectrometry. Identification of the peptide sequences and their post translational status can be performed before a protein binding assay is conducted. Alternatively identification of the peptide sequences and their post translational status can be performed before a protein binding assay is conducted. As well identification of the peptide sequences and their post translational status can be performed both before and after a protein binding assay is conducted.

The method includes screening a peptide library. In one example the method provides for the use of a one bead one compound OBOC peptide library also known as OBOC combinatorial library. Examples of such libraries are described in Lam et al. 1991 354 82 84 and in Furka et al. 199137 487 493.

The OBOC combinatorial library method synthesizes 10 10of random compounds such that each bead displays only one compound. Bead libraries are screened and positive beads are isolated for structure analysis. Peptide substrates and inhibitors of protein kinases and peptide ligands for cell surface receptors can be identified using this method Lam et al. 2003 36 370 377 .

A peptide library can be synthesized on various types of solid supports using methods known in the art for example those described in U.S. Pat. Nos. 7 122 323 and 5 510 240. Preferably the library is synthesized using beads as solid phase support. In one example the solid support can have the form of beaded resin beads .

The peptide library can be an OBOC acetyl peptide library. One or more of the amino acids from the peptide can be deprotected using methods known in the art e.g. trifluoroacetic acid. One or more of the amino acids from the peptide can be acetylated. In addition to acetylation of at least one amino acid one or more of the amino acids can be modified in a variety of ways for example through covalent modifications. Modifications can include post translational modifications or introduction of non classical amino acids as described in U.S. Pat. No. 5 510 240.

The length of the peptide chain can vary. Preferably the OBOC acetyl peptide library is generated using 5 mer peptides i.e. peptides with 5 amino acid residues. One or more of these 5 amino acids from the peptide can be acetylated. Preferably the central third amino acid is acetylated. More preferably the central acetylated amino acid is lysine. Preferably unique sequences are constructed around a central epsilon amino acetylated lysine.

When the peptide library is an acetyl peptide library the method can be used for determining protein deacetylase substrate specificity. The protein deacetylase can belong to the sirtuin family. Preferably the protein deacetylase is SIRT1.

The peptide library can be a combinatorial peptide library based on N terminal histone sequences. In this example one or more of the peptide sequences include N terminal histone sequences. Preferably the combinatorial OBOC peptide library is based on N terminal amino acid sequence of histone H4. The N terminal histone sequences can have a different number of amino acid residues. Preferably the N terminal histone sequences are 21 mers i.e. they have 21 amino acid residues. These sequences can be attached to solid support directly or via one or more amino acids that act as linkers.

The N terminal histone sequences can be modified to include various post translational modifications. The modifications can be covalent. The modifications can include for example methylation acetylation phosphorylation ubiquitination sumoylation citrullination or ADP ribosylation. Modifications can also include the introduction of non classical amino acids. Each N terminal histone sequence can include one or more of these modifications.

When the combinatorial peptide library is based on N terminal histone sequences the invention provides methods for determining and evaluating histone binding proteins or histone modifying enzymes. Preferably the histone binding protein is an antibody. More preferably the histone binding protein is an phos S1 H4 antibody.

Prior to library construction peptide length may initially be considered to determine whether relatively short peptides would function as efficient protein substrates. For example prior to library construction peptide length may be considered to determine whether relatively short acetyl peptides would function as efficient enzymatic substrates. Also for example prior to library construction peptide length may be considered to determine the preferred length of the N terminal amino acid chain of a histone and its efficiency as a protein substrate.

The method provides for the use of a quantum dot tagging i.e. labeling strategy. Quantum dots are nanoparticles that exhibit exquisite photochemical properties owing to their semiconductor cores and are emerging as ideal fluorophores for screening OBOC libraries Falciani et al. 2005 12 417 426 . These properties include robust photostability high quantum yield and a sharp emission with a broad range of excitation wavelengths. Coupled with a bead sorting instrument quantum dots allow the screening of hundreds of thousands of peptide sequences for protein binding and protein activity in a single day. Quantum dots can be used for example to label the protein that is in contact with a peptide that is attached to a bead.

The method provides for sorting of beads. Bead sorting can be performed manually or it can be automated. Bead sorting is preferably performed based on beads that are labeled. Bead labeling can be performed using a variety of methods known in the art. Preferably beads can be labeled with a fluorescent label or with any other type of label. When beads are labeled with a fluorescent label then the method can include fluorescent bead sorting. In one example a protein that is assayed and that specifically binds to beads can be labeled. In another example beads with peptides deacetylated by a protein deacetylase can be labeled. For example these beads can be first biotinylated and then tagged labeled with streptavidin coated quantum dots.

After bead sorting peptides sequences and their corresponding post translationally modified status can be extracted from individual beads in the library. The sequences and modification state of these peptides can then be identified. Identification of the peptides can be performed for example by mass spectrometry or by micro sequencing. Identification of the peptides can thus identify the particular sequences and modification state for which the assayed protein shows considerable binding preference.

A peptide library that is already spatially addressed e.g. each sequence and modification state is known at each support e.g. bead can be screened by the methods described.

The present invention provides a high throughput method for determining substrate specificity of protein deacetylases using a one bead one compound OBOC acetyl peptide library with a quantum dot tagging strategy and automated bead sorting. The OBOC acetyl peptide library method allows context specific identification of preferred peptide substrates. This is in contrast to the previously published approach that can only uncover globally preferred amino acids at each position. The OBOC acetyl peptide library method can be applied to any histone protein deacetylase from class I II or III.

Various applications of peptide libraries of this type can be envisioned. For example the sequence information obtained from this library can be used to generate acetyl peptide specific antibodies for Western blot analysis. This can provide in vivo validation of acetylation at protein acetylation sites discovered in BLAST searches and sequence comparisons. Such antibodies could also be employed in immunoprecipitation studies. Mass spectral analysis could then be performed to identify the acetylated proteins. Identification of enzymatic substrates i.e. the sequence information obtained from the library can also be used to generate acetyl peptide specific inhibitors.

Other uses of the library include the creation of super substrates for the in vivo generation of O acetyl ADP ribose OAADPr to elucidate its cellular roles. Co crystal studies could be executed to uncover how Sir2 interacts with these optimal substrate sequences. Limited peptide substrates co crystallized with Sir2 have shown interactions primarily with the peptide backbone Avalos et al. 2002 10 523 535 . However the method of this invention provides for the use of side chain interactions to bind and catalyze protein deacetylation.

Hits from libraries of this type could serve as starting points for the design of peptidomimetics for a variety of applications e.g. for use as potential therapeutics Nefzi et al. 2004 69 3603 3609 Falciani et al. 2005 12 417 426 . Optimized substrates reflect higher binding affinity to a protein e.g. enzyme. Modification of the peptide to prohibit enzymatic turnover and protease degradation could be implemented to generate a specific tight binding in vivo inhibitor.

The original peptide sequences can be further modified to confer altered chemical and biological properties Nefzi et al. 2004 69 3603 3609 Falciani et al. 2005 12 417 426 . This strategy has been used to tailor make peptides into therapeutics that avoid the pitfalls of proteolytic cleavage rapid clearance from the circulatory system inability to pass through the blood brain barrier and lack of oral activity Nefzi et al. 2004 .

This invention provides a method for identification of the molecular recognition events involved in the histone code via OBOC combinatorial peptide libraries based on N terminal histone sequences. This method provides for synthesis and evaluation of all possible permutations at known modification sites of the 21 N terminal amino acids of histone H4. In a preferred embodiment the evaluation of the binding specificity is performed using an antibody directed to serine phosphorylation and a quantum dot detection strategy Garske and Denu 2006 45 94 101 Kodadek et al. 2006 2 25 35 . However the general approach is applicable to interrogating the preferences of any histone binding protein or any histone modifying enzyme. As well this approach may find use in histone specific antibody screening.

All amino acid derivatives and resins were purchased from Peptides International Louisville Ky. or from Bachem Bubendorf Switzerland . Peptides used in the solution deacetylase assays were obtained from the University of Wisconsin Madison Biotechnology Core Facility. Other chemical reagents were obtained from Sigma Aldrich St. Louis Mo. Acros Geel Belgium Novabiochem San Diego Calif. Amersham Biosciences Buckinghampshire England or Quantum Dot Hayward Calif. . Reaction vessels for peptide library synthesis were purchased from Alltech Chromatography Deerfield Ill. .

Analytical gradient HPLC was conducted on a Shimadzu series 2010C HPLC with a Vydac C18 column 10 m 4.6 250 mm . All runs used linear gradients of 0.05 aqueous TFA and 0.02 TFA in acetonitrile. MALDI TOF MS was performed on a Bruker REFLEX II using cyano 4 hydroxy cinnamic acid as matrix. Fluorescent bead sorting was carried out on a COPAS Select Union Biometrica Holliston Mass. instrument. Fluorescence microscopy was done on an Olympus IX81 instrument Tokyo Japan equipped with a Hamamatsu digital camera Hamamatsu City Japan .

SIRT1 SIRT2 and ySir2 were expressed and purified as previously described Borra et al. 2004 43 9877 9887 Borra et al. 2005 280 17187 17195 .

Prior to kinetic analysis peptide concentrations were established by amino acid analysis AAA or by a coupled assay in which NAD leftover from exhaustive deacetylation reactions acetylated peptide was typically incubated with 5 10 M Sir2 and 80 M NAD for 20 min was quantitatively converted to NADH with alcohol dehydrogenase and monitored spectrophotometrically in real time at 340 nm. Peptide concentrations were obtained by subtracting the amount of NADH formed from the original amount of NAD used in the reaction.

All solution phase Sir2 assays were carried out at 25 C. in 50 mM Tris.HCl. Reactions were done in 50 100 L with 0.1 1.5 M enzyme 0.1 1.2 mM NAD 0.5 1000 M peptide and 1 mM DTT. Reaction mixtures were quenched with TFA to a final concentration of 1 after 5 10 min and nicotinamide levels were quantitated by HPLC at 264 nm.

Alternatively P NAD 10 mCi mL was used in assays and quenched reaction mixtures were spotted on a silica TLC plate and run in a chamber containing 60 ethanol and 40 2.5 mM ammonium acetate for 3 4 hours. Levels of P OAADPr and P NAD were then quantitated by phosphorimaging and the fraction turnover was calculated. Saturation curves were done at varying concentrations of peptide while holding that of NAD constant. Time points were chosen such that product formation never exceeded 20 conversion and data were plotted as rate s vs. peptide concentration. Plots were fitted to the Michaelis Menten equation v k K S 1 S K using Kaleidagraph software Reading Pa. to extract Kand k K.

The screening strategy used the reaction of biotin N hydroxy succinimide ester with the newly generated amino group formed upon SIRT1 deacetylation. Subsequent binding of the streptavidin conjugated quantum dots provided the fluorescent tag for screening.

Initially it was established that quantum dot labeling was proportional to the molar abundance of reacted biotin. Resin bearing free amino groups were aliquoted into five reaction vessels and labeled with 1 0.5 0.01 0.001 and 0 equivalents of biotin N hydroxy succinimide ester. After differential labeling the resin was pooled into reaction vessel and a streptavidin conjugated quantum dot 605 nm solution was added. After draining the quantum dot solution and washing the resin the resulting pooled beads displayed differential levels of associated quantum dots correlating with the amount of covalently linked biotin.

To provide a quantitative assessment of fluorescent quantum dot labeling a complex object parametric analyzer and sorter COPAS instrument was utilized. COPAS sorts beads based on fluorescence intensity while also gathering data on bead size time of flight . Using this instrument beads labeled in the previously mentioned experiment were sorted with an excitation of 488 nm and an emission of 610 nm. The fluorescence distribution was plotted and distinct populations could be visualized . depicts a log scale plot of fluorescence intensity vs. TOF bead size for quantum dot labeled beads with biotinylation levels of 1 0.5 0.01 0.001 and 0 equivalents as analyzed by the COPAS beadsorter at 610 nm.

These populations resided in a fluorescence regime that encompassed more than two orders of magnitude. Moreover the bead groupings corresponded to the differential levels of biotinylation although the 0.001 and 0 biotin equivalents coalesced into a single cluster . When the emission wavelength was set to green light 510 nm corresponding to the intrinsic TentaGel autofluorescence only a single population was observed. Quantum dot labeling was found to be quantitative in sub stoichiometric amounts. It can be coupled to a sorting instrument such as the COPAS instrument for sorting beads on the basis of fluorescence intensity.

Prior to library construction it was essential to determine whether relatively short acetyl peptides would function as efficient substrates of Sir2 enzymes. To evaluate peptide length requirements 10 acetyl lysine containing peptides corresponding to the histone H3 sequence surrounding Lys 14 and of varying length were assayed using SIRT1 and a variety of other sirtuins SIRT2 yeast Sir2 ySir2 and Sir2 TbSir2 . Deacetylation assays were conducted at fixed NAD concentrations while peptide concentrations were varied to produce saturation curves. The resulting data were fitted to the Michaelis Menten equation to yield catalytic efficiencies as defined by the apparent second order rate constant k K which takes into consideration both binding and catalysis. All peptides used in these studies were N terminally acetylated but the N terminus was not deacetylated by sirtuins in control assays.

The results shown in Table 1 are represented as relative k Kvalues with the longest peptide AcTGG AcK APRK SEQ ID NO 9 given a value of one. In these studies all sirtuins surveyed showed no more than a 2 to 3 fold difference in k Kfor the various peptide substrates. Thus the shortest peptide a 5 mer was similar in catalytic efficiency to the longest peptides in this preliminary set regardless of the enzyme assayed. These observations suggest amino acids beyond the 2 and 2 positions are not necessary for efficient binding and catalysis by sirtuins. For library construction balancing minimal peptide length with practical limitations of library complexity were important considerations. Consequently a 5 mer library with an acetylated lysine residue in the central position was constructed.

After validating that quantitative quantum dot labeling could be used in conjunction with fluorescence based bead sorting an OBOC peptide library was constructed using the split pool method Lam et al. 1991 354 82 84 Furka et al. 1991 37 487 493 . Eighteen variable amino acids were used at four positions centered around an acetylated lysine two amino acids on each side of the acetylated lysine residue . All common natural amino acids excluding cysteine lysine methionine and arginine were used. To mimic charged residues dimethyl arginine was substituted for lysine and arginine. To avoid unwanted cyanogen bromide cleavage points isosteric norleucine was used in place of methionine. Lysine and cysteine were not included in the library because both residues would produce false hits in addition to the problems posed by disulfide formation in the latter case because the nucleophilicity of the amine and sulfhydryl groups respectively would result in their biotinylation and subsequent quantum dot labeling.

The acetylated peptide library was constructed on TentaGel Macrobead NHresin 280 320 m 0.21 mmol g loading 65 550 beads g using the split pool approach. Fmoc tBu methodology Bodanszky 1993 2nd ed. Springer Verlag Germany was used to carry out the library synthesis on 4.80 g of resin. Prior to randomization a four amino acid linker BBRM B alanine was synthesized. After deprotecting the N terminus with 20 v v piperidine in DMF for 15 min the resin was split equally into eighteen separate reaction vessels one for each amino acid in the library . Four equivalents of amino acid and coupling reagent in addition to 5 mol mol capping reagent were added to each vessel for later sequencing.

Capping reagents included phenylacetic acid and 4 pentenoic acid . Phenylacetic acid was used in conjunction with norleucine while 4 pentenoic acid was used with all other amino acids. Equimolar ratios of both capping reagents were used for isoleucine asparagine glutamine and histidine. After a second coupling the resin from all vessels was washed three times each with DCM and DMF pooled and deprotected. Next the resin was redistributed into the reaction vessels for coupling of the second randomized residue. The process was repeated and after pooling N acetyl lysine was installed as the third residue with no capping . The split pool technique was repeated for the fourth and fifth randomized residues. After the final N terminal deprotection the N termini of all the peptides were acetylated 70 DCM 25 acetic anhydride 5 triethylamine and washed with DCM. Reagent K TFA EDT thioanisole water phenol 82.5 2.5 5 5 5 King et al. 36 255 266 was used as the global deprotection cocktail. The resin was washed thoroughly with DCM and stored at 20 C. until use.

In preliminary studies incorporation of arginine residues beyond the linker position gave false positive signals in the on bead assays due to reaction with the biotin ester during the labeling step. This was an unfortunate result as it precluded incorporation of arginine in the library. The same problems posed by the reactivity of arginine have prevented its incorporation in a previous library Hu et al. 1999 38 643 650 . To mimic positively charged residues lysine and arginine symmetrical dimethyl arginine was used. Thus 18 104 907 sequences were represented in the library. A threefold excess of beads was used to give 95 probability that all sequences were represented Altschul et al. 1997 25 3389 3402 .

After library synthesis the on bead SIRT1 deacetylation assay was carried out as shown in . In this assay all beads were simultaneously subjected to deacetylation conditions 0.35 M SIRT1 12 min at 25 C. allowing competition of all peptide sequences for reaction with SIRT1. Afterwards the beads were washed and subjected to biotinylation conditions in DMF. Excess reagent was removed prior to blocking non specific protein binding sites with BSA and subsequent quantum dot labeling. Lastly beads were washed a final time and sorted using a COPAS instrument to obtain data such as those shown in .

Determining the viability of quantitative quantum dot analysis was also performed. Five 10 mg portions of TentaGel S NHresin 90 m 0.26 mmol g loading 2.86 10beads g were divided out and swollen in DCM. After washing with DCM 3 1 mL and DMF 3 1 mL the beads were labeled with 1 0.5 0.01 and 0.001 molar equivalents of N hydroxysuccinimidobiotin in 200 L portions of DMF. After an hour of rocking at room temperature the solutions were drained and washed with DMF 3 1 mL . Approximately 5 mg of resin from each of the above reactions were combined and incubated with 1 mL BSA 1 mg mL in TBST buffer 25 mM Tris.HCl pH 8.0 150 mM NaCl and 0.1 Tween 20 for 1 hour. The resin was washed with TBST buffer 3 1 mL and drained to the level of the resin bed. At this point 500 L of 75 nM streptavidin coated Q dot 605 in TBST buffer was poured over the resin and rocked for 2 hours after which the solution was drained to the resin bed before washing with TBST 10 1 mL . Beads were photographed using a fluorescence microscope with a FITC filter and sorted on the basis of fluorescence 488 nm 610 nm with a COPAS Select sorting instrument. Sorting data were evaluated with FCS Express De Novo Software Thornhill Ontario in histogram and dot plot form.

The entire library was assayed in a 75 mL column equipped with a filter. Prior to the assay the resin was sequentially washed with DCM 3 50 mL DMF 3 50 mL and deacetylation assay buffer 50 mM Tris pH 7.5 1 50 mL . The enzymatic reaction was initiated upon addition of 50 mL of deacetylation cocktail 0.35 M SIRT1 1.5 mM NAD 1 mM DTT . The reaction mixture was allowed to rock gently for 12 min. After draining the resin was washed with doubly distilled water 5 50 mL and DMF 5 50 mL . Afterwards the resin was rocked with biotin N hydroxy succinimide ester in DMF 3.5 mM 50 mL for 20 min. The solution was drained and the resin was washed with DMF 6 50 mL and TBST buffer 2 50 mL .

To reduce nonspecific binding the beads were incubated with 50 mL of BSA 2 mg mL in TBST buffer for 1.5 hours. After draining and washing with TBST buffer 1 50 mL 50 mL of 4 nM streptavidin coated Q Dot 605 in TBST buffer was added and the mixture was allowed to rock for 2 hours. Again the solution was drained and washed with TBST buffer 10 50 mL . The resin was then suspended in a minimal amount of TBST buffer and refrigerated at 4 C. overnight.

In order to extract peptide sequences from individual beads in the library a previously developed capping method was improved in which sequence decoding is done by reading a mass spectral peptide ladder Youngquist et al. 1995 117 3900 3906 . Instead of using the acetyl group for capping during peptide synthesis two carboxylic acids were used phenylacetic acid 1 and 4 pentenoic acid 2 . First a four amino acid linker was synthesized onto TentaGel beads to extend the bound peptide into solution and to bring the peptide mass out of the MALDI matrix region. This linker was composed of methionine for a cyanogen bromide cleavage point arginine for improved mass spectral analysis and two alanines for added flexibility . This capping method which utilizes two caps allows for the identification of up to three isobaric i.e. of identical mass amino acids.

In each coupling step of a randomized residue a small amount of capping reagent was added to terminate chain growth for later sequencing . In each capping step either one or both of the capping reagents were used. The use of two reagents assisted in deciphering amino acids of similar or identical masses. In cases where both caps were used a signature doublet would appear on the mass spectrum. By HPLC analysis it was determined that 5 mol capping at each step in the synthesis of a prototypical 5 mer yielded 79 full length peptide. This amount of capping reagent provided a more than adequate amount for on bead assay yet produced enough capped material to produce quality peptide ladders in the mass spectra. An acetyl group served as the N terminal cap.

Beads were sorted on the basis of fluorescence 488 nm 610 nm using the COPAS instrument. Initially the 300 most intensely fluorescent beads 0.1 were collected pooled and then sorted a second time to generate an enriched sample of the 45 brightest beads. After washing in a guanidinium hydrochloride solution single beads were placed in separate microcentrifuge tubes and treated overnight with a cyanogen bromide cleavage cocktail. The cleavage products were subsequently subjected to MALDI TOF MS for sequence analysis . Of those 45 beads 33 were sequenced successfully from their mass spectra Table 2 6 were found to be damaged and were not sequenced while the remaining 6 yielded spectra that were not interpretable. BLAST searches of the mammalian proteome were performed in the short nearly exact mode for the 33 sequences obtained from the library see Table 4 .

Shown in Table 2 are peptide sequences of hits from the SIRT1 combinatorial library screen. Position 2 is the N terminal end and Position 2 is the C terminal end. Uncertainty in the order of N terminal and adjacent amino acids is signified by the symbol .

Beads from the enriched sample were pooled and washed with 8 M guanidinium hydrochloride 2 1 mL doubly distilled water 10 1 mL and DMF 3 1 mL . Individual beads were then deposited into separate microcentrifuge tubes containing 20 L of cleavage cocktail 70 TFA 30 doubly distilled water and 20 cyanogen bromide by weight as described by Hu et al. 1999 . After incubation overnight in the dark the samples were dried and resuspended in 5 L of 0.1 TFA. Each sample 1 L was combined with saturated matrix solution 1 L and dried on the target for MALDI TOF MS analysis positive ion mode .

To validate the results of the library screen select hits and non hits were resynthesized and subjected to in solution kinetic analysis Table 3 . A radioactive TLC based assay was employed with subsaturating levels of P NAD to determine the relative catalytic efficiencies Jackson et al. 2003 278 50985 50998 . In addition two consensus peptides containing residues occurring with the highest and lowest frequency at each position independent of context were analyzed. For comparison a 5 mer comprised of a sequence corresponding to a known site for p53 deacetylation by SIRT1 was assayed Table 3 .

In Table 3 efficiencies average were obtained by fitting the data from P NAD assays to the modified Michaelis Menten equation v k K S 1 S K . No definite catalytic efficiency for VQ AcK II SEQ ID NO 31 was established due to problems with insolubility a lower limit was established. Catalytic efficiencies of peptides containing the residues of the highest lowestfrequency at each position and the sequence relevant to p53 deacetylation in vivoare shown for comparison.

Hit sequences had significantly higher catalytic activity than non hits. Some hits were near or greater than an order of magnitude more catalytically active than their non hit counterparts. Hits correlated with increased catalytic activity by as much as 20 fold. Most hit sequences assayed in solution had significantly higher activity than the peptide sequence relevant to in vivo p53 deacetylation. One non hit sequence AY AcK EV SEQ ID NO 19 had a catalytic activity comparable to those of a few of the hits.

Although the apparent second order rate constant k K varied widely among the peptides tested the turnover number k was relatively constant at 0.1 s. Differences in k kreflect differences in peptide binding affinity.

One of the main advantages of this OBOC library is its context specific nature. In other words there is no implicit assumption that residues in substrate sequences function independently of one another. While oriented peptide libraries can be useful in resolving globally preferred consensus sequences Songyang et al. 1994 Blander et al. 2005 they do not provide contextual information.

The so called consensus peptides WH AcK QQ SEQ ID NO 22 and WP AcK QQ SEQ ID NO 23 show a seven fold difference in catalytic activity in favor of WH AcK QQ SEQ ID NO 22 Table 3 . Thus in the context of WX AcK QQ SEQ ID NO 72 a histidine is greatly preferred at position 1. Within the XP AcK QX context SEQ ID NO 73 QP AcK QI SEQ ID NO 30 is favored over WP AcK QQ SEQ ID NO 23 by 18 fold. Thus SIRT1 mediated deacetylation is stringently context dependent and that there is no best average sequence . Further support comes from the fact that although proline residues at 1 are not well tolerated when adjacent to a tryptophan at 2 they appear to function well when adjacent to dimethyl arginine at 2. There are synergistic antagonistic relationships among certain residues and that this plays a significant role in substrate recognition by SIRT1.

BLAST searches of the SIRT1 hits Table 2 within the mammalian proteome reveal correspondence to a number of proteins Table 4 some of which are known to be acetylated in vivo.

An OBOC histone H4 N terminal tail combinatorial library was constructed to identify the binding preferences of the antibody toward all known possible histone modification states. The H4 histone tail library was comprised of the sequence corresponding to the first 21 amino acids of human histone H4 attached to a linker composed of 2 alanines B and a methionine M . The library included 800 unique peptide sequences representing all known modification states for the first 21 amino acids of histone H4 in addition to all possible methylation states at lysines and arginines that are known to be methylated. Using an phos S1 H4 antibody as a primary antibody the library was screened to determine histone H4 N terminal sequences to which the primary antibody specifically bound.

Amino acid derivatives and resins were purchased from Peptides International Louisville Ky. Novabiochem San Diego Calif. or from Bachem Bubendorf Switzerland . Other chemical reagents were obtained from Sigma Aldrich Invitrogen Carlsbad Calif. or Jackson ImmunoResearch Laboratories West Grove Pa. . The phos S1 H4 antibody was a gift from the laboratory of C. David Allis Rockefeller University New York N.Y. . Peptides were synthesized on a Symphony synthesizer from Protein Technologies Tucson Ariz. . Filter columns for on bead assays were obtained from Alltech Deerfield Ill. .

Analytical gradient HPLC was performed on a Shimadzu series 2010C HPLC with a Vydac C18 column 10 m 4.6 250 mm . All runs employed linear gradients of 0.05 aqueous TFA and 0.02 TFA in acetonitrile. Microextraction tips for desalting peptides were purchased from Varian Inc. Palo Alto Calif. . MALDI TOF MS was performed on a Bruker REFLEX II and MALDI TOF TOF MS was executed on an Applied Biosystems 4800. A Zeiss Axioplan 2 microscope Jena Germany with a DAPI dye bandpass filter 390 410 nm and an AxioCam MRm was used for fluorescence microscopy.

Five mg quantities of TentaGel Macrobead NHresin 280 320 m 0.27 mmol g loading 65 550 beads g bearing either a phosphorylated or unphosphorylated histone H4 sequence or a mixture were added to 1.5 mL filter columns washed thoroughly with DCM MeOH doubly distilled water ddHO and PBST buffer 25 mM NaPi pH 7.4 150 mM NaCl and 0.1 Tween 20 . The resin was swelled for 1 hour with gentle rocking prior to drainage and one hour of blocking with 3 w v bovine serum albumin BSA in PBST. After draining the blocking solution to the resin bed 100 L of a 100 1 dilution of phos S1 H4 antibody in PBST containing 3 BSA was added and the resin was allowed to rock gently for one hour.

After draining to the resin bed the resin was washed 3 100 L PBST and 100 L of 50 nM biotinylated goat anti rabbit antibody in PBST containing 3 BSA was added. One hour of gentle rocking was followed by draining the solution to the resin bed and washing 3 100 L PBST.

The resin was incubated with 100 L of 25 nM solution of Q dot 605 streptavidin conjugate in PBST and gently rocked for 2 hours. Following drainage to the resin bed the resin was washed 10 200 L PBST. The resin was then resuspended in PBST and viewed under a fluorescence microscope.

The combinatorial histone H4 peptide library was constructed on TentaGel Macrobead NHresin 280 320 m 0.27 mmol g loading 65 550 beads g using the split pool approach Lam et al. 1991 354 82 84 Furka et al. 1991 37 487 493 for sites of variability. Sites of variability include positions 20 K AcK MeK MeK MeK 16 K AcK 12 K AcK 8 K AcK 5 K AcK 3 R MeR MeR MeR citrulline and 1 S pS . The synthesis was performed on a 50 mol scale with standard Fmoc tBu chemistry Bodanszky M. 1993 2nd ed. Springer Verlag Germany . All amino acids at least 4.7 equivalents coupling were double coupled for 2 hour time periods.

Prior to the partially randomized histone H4 sequence a 3 amino acid linker BBM where B alanine M methionine was synthesized. After the final N terminal deprotection the N termini of all the peptides were acetylated with acetic anhdyride. A 50 mg 13.5 mol portion of the library was deprotected for 5 hours with Reagent K TFA EDT thioanisole water phenol 82.5 2.5 5 5 5 prior to use as described by King et al. 1990 36 255 266. The remainder of the library was stored at 4 C. for later use.

Twenty beads were randomly selected from the library and deposited into separate microcentrifuge tubes containing 20 L of cleavage cocktail 70 TFA 30 ddHO and 20 cyanogen bromide by weight Hu et al. 1999 38 643 650 . After incubation overnight in the dark the samples were dried.

Ten of the cleavage products were desalted and sequenced by MALDI TOF TOF MS. The remaining 10 cleavage products were dissolved in 100 L quantities of ddHO and analyzed by analytical RP HPLC. Fractions corresponding to the primary peak at 214 nm were lyophilized and resuspended in 5 L of ddHO. Each sample 1 L was combined with saturated matrix solution 1 L and dried on the target for MALDI TOF MS analysis positive ion mode .

Prescreen was performed by adding 50 mg 13.5 mol of the peptide library to a 4 mL filter column and washing it thoroughly with DCM MeOH ddHO and PBST buffer. The resin was swelled for 1 hour with gentle rocking prior to drainage and one hour of blocking with 3 BSA in PBST. After draining the blocking solution to the resin bed 1 mL of 50 nM biotinylated goat anti rabbit antibody in PBST containing 3 BSA was added. Following 1 hour of rocking the solution was drained to the resin bed and washed 3 1 mL PBST. The resin was then rocked with 1 mL of a 25 nM solution of Q dot 605 streptavidin conjugate in PBST for two hours. Following drainage to the resin bed the resin was washed 10 2 mL PBST. At this point the resin was resuspended in PBST and viewed under a fluorescent microscope and any fluorescent beads could be removed from the library.

After prescreening the library for nonspecific interactions with the secondary antibody or the quantum dots a screen was performed. The only difference from the prescreen was a one hour incubation with 1 mL of a 100 1 dilution of phos S1 H4 in PBST with 3 BSA after the swell step and washing 3 1 mL PBST prior to addition of the secondary antibody. When viewed under the microscope a number of brightly fluorescent moderately fluorescent and dark beads were manually selected.

Beads that were selected under the microscope were incubated with 200 L of 8 M guanidinium hydrochloride prior to washing 3 500 L ddHO and drying. Peptides were cyanogen bromide cleaved from each bead and desalted before sequencing with MALDI TOF TOF MS.

After demonstrating detection of serine phosphorylation of a histone H4 sequence immobilized on a bead an OBOC histone H4 combinatorial library H4 histone N terminal tail library was constructed to further examine the binding preferences of the antibody toward all known possible histone modification states. The H4 histone tail library schematically shown in is comprised of the sequence corresponding to the first 21 amino acids of human histone H4 attached to a linker composed of 2 alanines B and a methionine M . Sites for modification include positions 20 16 12 8 5 3 and 1 and are annotated with X. Possible modification states are shown above or below the peptide chain . In this library schematically shown in the first 21 amino acids of histone H4 are represented with 7 sites of variability. The sites include positions 20 K AcK MeK MeK MeK 16 K AcK 12 K AcK 8 K AcK 5 K AcK 3 R MeR MeR MeR citrulline and 1 S pS . This library includes all known modification states for the first 21 amino acids of histone H4 in addition to all possible methylation states at lysines and arginines that are known to be methylated. Therefore this library is composed of 800 unique peptide sequences with 99 confidence of 95 coverage of combinatorial space each library consists of 50 mg of resin Burgess et al. 37 2985 2987 .

The library synthesis was followed by rigorous evaluation of the synthetic product. RP HPLC analysis of the cleavage products from 10 individual beads revealed peptides of 90 95 purity within the correct mass range. In addition the cleavage products from 10 additional randomly selected beads were successfully sequenced with MALDI TOF TOF MS.

The library was first prescreened with only the secondary antibody and quantum dots. The fact that none of the beads exhibited fluorescence due to quantum dots suggested the absence of non specific interactions between the immobilized peptides with either the secondary antibody or the quantum dots. Therefore when the primary antibody was included in a screening experiment the fluorescence observed was due to a specific interaction with the primary antibody . Of the library about half of the beads exhibited some level of quantum dot associated fluorescence. A number of individual beads of were manually selected and classified as either fluorescent moderately fluorescent or dark.

Data from the screen indicate the binding preferences of the phos S1 H4 antibody for certain sequences Table 5 . Twenty beads were manually selected from a screen of a histone H4 tail library. Sequences were elucidated by MALDI TOF TOF MS. All sequences obtained from fluorescent beads were phosphorylated while the moderately fluorescent beads displayed peptides that were typically phosphorylated and generally highly modified . Eighty percent of the dark beads harbored peptides that were unphosphorylated. Legend pS phosphorylated serine AcK acetylated lysine MeK MeK MeK the correspondingly methylated states of lysine MeR and MeR the correspondingly methylated states of arginine where a and s refer to symmetric and asymmetric respectively.

It is to be understood that this invention is not limited to the particular devices methodology protocols subjects or reagents described and as such may vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only and is not intended to limit the scope of the present invention which is limited only by the claims. Other suitable modifications and adaptations of a variety of conditions and parameters normally encountered in clinical prevention and therapy obvious to those skilled in the art are within the scope of this invention. All publications patents and patent applications cited herein are incorporated by reference in their entirety for all purposes.

