---

title: Nucleic acid and amino acid sequences relating to for diagnostics and therapeutics
abstract: The invention provides isolated polypeptide and nucleic acid sequences derived from that are useful in diagnosis and therapy of pathological conditions; antibodies against the polypeptides; and methods for the production of the polypeptides. The invention also provides methods for the detection, prevention and treatment of pathological conditions resulting from bacterial infection.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07626000&OS=07626000&RS=07626000
owner: Sanofi Pasteur Limited/Sanofi Pasteur Limit√©e
number: 07626000
owner_city: Toronto, Ontario
owner_country: CA
publication_date: 20061220
---
This application is a divisional of U.S. application Ser. No. 11 028 458 now U.S. Pat. No. 7 378 258 filed Dec. 30 2004 which is a continuation of U.S. application Ser. No. 10 640 833 now abandoned filed Aug. 14 2003 which is a continuation of U.S. application Ser. No. 09 583 110 now U.S. Pat. No. 6 699 703 filed May 26 2000 which is a continuation in part of U.S. application Ser. No. 09 107 433 now U.S. Pat. No. 6 800 744 filed Jun. 30 1998 which claims the benefit of U.S. Application No. 60 085 131 filed May 12 1998 and of U.S. Application No. 60 051 553 filed Jul. 2 1997. The entire teachings of the above applications are incorporated herein by reference.

This application incorporates by reference the Sequence Listing contained on the two compact disks Copy 1 and Copy 2 filed concurrently herewith containing the following file 

This application also incorporates by reference Table 2 contained on the two compact disks Copy 1 and Copy 2 filed concurrently herewith containing the following file 

The invention relates to isolated nucleic acids and polypeptides derived from that are useful as molecular targets for diagnostics prophylaxis and treatment of pathological conditions as well as materials and methods for the diagnosis prevention and amelioration of pathological conditions resulting from bacterial infection.

Vaccines against have been available for a number of years. There are a large number of serotypes based on the polysaccharide capsule van Dam J. E. Fleer A. and Snippe H. 1990 Antonie van Leeuwenhoek 58 1 although only a fraction of the serotypes seem to be associated with infections Martin D. R. and Brett M. S. 1996 N. Z. Med. J. 109 288 . A multivalent vaccine against capsular polysaccharides of 23 serotypes Smart L. E. Dougall A. J. and Gridwood R. W. 1987 J. Infect. 14 209 has provided protection for some groups but not for several groups at risk for pneumococcal infections such as infants and the elderly M kel P. H. et al. 1980 Lancet 2 547 Sankilampi U. 1996 J. Infect. Dis. 173 387 . Conjugated pneumococcal capsular polysaccharide vaccines have somewhat improved efficacy but are costly and therefore are not likely to be in widespread use K yhty H. and Eskola J. 1996 Emerg. Infect. Dis. 2 289 .

At one time strains were uniformly susceptible to penicillin. The report of a penicillin resistant strain of Hansman D. and Bullen M. M. 1967 Lancet 1 264 was followed rapidly by many reports indicating the worldwide emergence of penicillin resistant and penicillin non susceptible strains Klugman K. P. 1990 Clin. Microbiol. Rev. 3 171 . strains which are resistant to multiple antibiotics including penicillin have also been observed recently within the United States Welby P. L. 1994 Pediatr. Infect. Dis. J. 13 281 Ducin J. S. et al. 1995 Pediatr. Infect. Dis. J. 14 745 Butler J. C. 1996 J. Infect. Dis. 174 986 as well as internationally Boswell T. C. et al. 1996 J. Infect. 33 17 Catchpole C. Fraise A. and Wise R. 1996 Microb. Drug Resist. 2 431 Tarasi A. et al. 1997 Microb. Drug Resist. 3 105 .

A high incidence of morbidity is associated with invasive infections Williams W. W. et al. 1988 Ann. Intern. Med. 108 616 . Because of the incomplete effectiveness of currently available vaccines and antibiotics the identification of new targets for antimicrobial therapies including but not limited to the design of vaccines and antibiotics which may help prevent infection or that may be useful in fighting existing infections is highly desirable.

The present invention fulfills the need for diagnostic tools and therapeutics by providing bacterial specific compositions and methods for detecting treating and preventing bacterial infection in particular infection.

The present invention encompasses isolated polypeptides and nucleic acids derived from that are useful as reagents for diagnosis of bacterial infection components of effective antibacterial vaccines and or as targets for antibacterial drugs including anti drugs. The nucleic acids and peptides of the present invention also have utility for diagnostics and therapeutics for and other species. They can also be used to detect the presence of and other species in a sample and in screening compounds for the ability to interfere with the life cycle or to inhibit infection. More specifically this invention features compositions of nucleic acids corresponding to entire coding sequences of proteins including surface or secreted proteins or parts thereof nucleic acids capable of binding mRNA from proteins to block protein translation and methods for producing proteins or parts thereof using peptide synthesis and recombinant DNA techniques. This invention also features antibodies and nucleic acids useful as probes to detect infection. In addition vaccine compositions and methods for the protection or treatment of infection by are within the scope of this invention.

The nucleotide sequences provided in SEQ ID NO 1 SEQ ID NO 2661 a fragment thereof or a nucleotide sequence at least 99.5 identical to a sequence contained within SEQ ID NO 1 SEQ ID NO 2661 may be provided in a variety of medias to facilitate use thereof. As used herein provided refers to a manufacture other than an isolated nucleic acid molecule which contains a nucleotide sequence of the present invention i.e. the nucleotide sequence provided in SEQ ID NO 1 SEQ ID NO 2661 a fragment thereof or a nucleotide sequence at least 99.5 identical to a sequence contained within SEQ ID NO 1 SEQ ID NO 2661. Uses for and methods for providing nucleotide sequences in a variety of media is well known in the art see e.g. EPO Publication No. EP 0 756 006 

In one application of this embodiment a nucleotide sequence of the present invention can be recorded on computer readable media. As used herein computer readable media refers to any media which can be read and accessed directly by a computer. Such media include but are not limited to magnetic storage media such as floppy discs hard disc storage media and magnetic tape optical storage media such as CD ROM electrical storage media such as RAM and ROM and hybrids of these categories such as magnetic optical storage media. A person skilled in the art can readily appreciate how any of the presently known computer readable media can be used to create a manufacture comprising computer readable media having recorded thereon a nucleotide sequence of the present invention.

As used herein recorded refers to a process for storing information on computer readable media. A person skilled in the art can readily adopt any of the presently known methods for recording information on computer readable media to generate manufactures comprising the nucleotide sequence information of the present invention.

A variety of data storage structures are available to a person skilled in the art for creating a computer readable media having recorded thereon a nucleotide sequence of the present invention. The choice of the data storage structure will generally be based on the means chosen to access the stored information. In addition a variety of data processor programs and formats can be used to store the nucleotide sequence information of the present invention on computer readable media. The sequence information can be represented in a word processing text file formatted in commercially available software such as WordPerfect and Microsoft Word or represented in the form of an ASCII file stored in a database application such as DB2 Sybase Oracle or the like. A person skilled in the art can readily adapt any number of data processor structuring formats e.g. text file or database in order to obtain computer readable media having recorded thereon the nucleotide sequence information of the present invention.

By providing the nucleotide sequence of SEQ ID NO 1 SEQ ID NO 2661 a fragment thereof or a nucleotide sequence at least 99.5 identical to a sequence contained within SEQ ID NO 1 SEQ ID NO 2661 in computer readable form a person skilled in the art can routinely access the sequence information for a variety of purposes. Computer software is publicly available which allows a person skilled in the art to access sequence information provided in a computer readable media. Examples of such computer software include programs of the Staden Package DNA Star MacVector GCG Wisconsin Package Genetics Computer Group Madison Wis. and NCBI toolbox National Center for Biotechnology Information .

Computer algorithms enable the identification of open reading frames ORFs within SEQ ID NO 1 SEQ ID NO 2661 which contain homology to ORFs or proteins from other organisms. Examples of such similarity search algorithms include the BLAST Altschul et al. J. Mol. Biol. 215 403 410 1990 and Smith Waterman Smith and Waterman 1981 Advances in Applied Mathematics 2 482 489 search algorithms. These algorithms are utilized on computer systems as exemplified below. The ORFs so identified represent protein encoding fragments within the genome and are useful in producing commercially important proteins such as enzymes used in fermentation reactions and in the production of commercially useful metabolites.

The present invention further provides systems particularly computer based systems which contain the sequence information described herein. Such systems are designed to identify commercially important fragments of the genome. As used herein a computer based system refers to the hardware means software means and data storage means used to analyze the nucleotide sequence information of the present invention. The minimum hardware means of the computer based systems of the present invention comprises a central processing unit CPU input means output means and data storage means. A person skilled in the art can readily appreciate that any one of the currently available computer based systems is suitable for use in the present invention. The computer based systems of the present invention comprise a data storage means having stored therein a nucleotide sequence of the present invention and the necessary hardware means and software means for supporting and implementing a search means. As used herein data storage means refers to memory which can store nucleotide sequence information of the present invention or a memory access means which can access manufactures having recorded thereon the nucleotide sequence information of the present invention.

As used herein search means refers to one or more programs which are implemented on the computer based system to compare a target sequence or target structural motif with the sequence information stored within the data storage means. Search means are used to identify fragments or regions of the genome which are similar to or match a particular target sequence or target motif. A variety of known algorithms are known in the art and have been disclosed publicly and a variety of commercially available software for conducting homology based similarity searches are available and can be used in the computer based systems of the present invention. Examples of such software include but is not limited to FASTA GCG Wisconsin Package Bic SW Compugen Bioccelerator BLASTN2 BLASTP2 and BLASTX2 NCBI and Motifs GCG . BLASTN2 A person skilled in the art can readily recognize that any one of the available algorithms or implementing software packages for conducting homology searches can be adapted for use in the present computer based systems.

As used herein a target sequence can be any DNA or amino acid sequence of six or more nucleotides or two or more amino acids. A person skilled in the art can readily recognize that the longer a target sequence is the less likely a target sequence will be present as a random occurrence in the database. The most preferred sequence length of a target sequence is from about 10 to 100 amino acids or from about 30 to 300 nucleotide residues. However it is well recognized that many genes are longer than 500 amino acids or 1.5 kb in length and that commercially important fragments of the genome such as sequence fragments involved in gene expression and protein processing will often be shorter than 30 nucleotides.

As used herein a target structural motif or target motif refers to any rationally selected sequence or combination of sequences in which the sequence s are chosen based on a specific functional domain or three dimensional configuration which is formed upon the folding of the target polypeptide. There are a variety of target motifs known in the art. Protein target motifs include but are not limited to enzymatic active sites membrane spanning regions and signal sequences. Nucleic acid target motifs include but are not limited to promoter sequences hairpin structures and inducible expression elements protein binding sequences .

A variety of structural formats for the input and output means can be used to input and output the information in the computer based systems of the present invention. A preferred format for an output means ranks fragments of the genome possessing varying degrees of homology to the target sequence or target motif. Such presentation provides a person skilled in the art with a ranking of sequences which contain various amounts of the target sequence or target motif and identifies the degree of homology contained in the identified fragment.

A variety of comparing means can be used to compare a target sequence or target motif with the data storage means to identify sequence fragments of the genome. In the present examples implementing software which implement the BLASTP2 and bic SW algorithms Altschul et al. J. Mol. Biol. 215 403 410 1990 Compugen Biocellerator was used to identify open reading frames within the genome. A person skilled in the art can readily recognize that any one of the publicly available homology search programs can be used as the search means for the computer based systems of the present invention.

The invention features polypeptides preferably a substantially pure preparation of an polypeptide or a recombinant polypeptide. In preferred embodiments the polypeptide has biological activity the polypeptide has an amino acid sequence at least 60 70 80 90 95 98 or 99 identical to an amino acid sequence of the invention contained in the Sequence Listing preferably it has about 65 sequence identity with an amino acid sequence of the invention contained in the Sequence Listing and most preferably it has about 92 to about 99 sequence identity with an amino acid sequence of the invention contained in the Sequence Listing the polypeptide has an amino acid sequence essentially the same as an amino acid sequence of the invention contained in the Sequence Listing the polypeptide is at least 5 10 20 50 100 or 150 amino acid residues in length the polypeptide includes at least 5 preferably at least 10 more preferably at least 20 more preferably at least 50 100 or 150 contiguous amino acid residues of the invention contained in the Sequence Listing. In yet another preferred embodiment the amino acid sequence which differs in sequence identity by about 7 to about 8 from the amino acid sequences of the invention contained in the Sequence Listing is also encompassed by the invention.

In preferred embodiments the polypeptide is encoded by a nucleic acid of the invention contained in the Sequence Listing or by a nucleic acid having at least 60 70 80 90 95 98 or 99 homology with a nucleic acid of the invention contained in the Sequence Listing.

In a preferred embodiment the subject polypeptide differs in amino acid sequence at 1 2 3 5 10 or more residues from a sequence of the invention contained in the Sequence Listing. The differences however are such that the polypeptide exhibits an biological activity e.g. the polypeptide retains a biological activity of a naturally occurring enzyme.

In preferred embodiments the polypeptide includes all or a fragment of an amino acid sequence of the invention contained in the Sequence Listing fused in reading frame to additional amino acid residues preferably to residues encoded by genomic DNA 5 or 3 to the genomic DNA which encodes a sequence of the invention contained in the Sequence Listing.

In yet other preferred embodiments the polypeptide is a recombinant fusion protein having a first polypeptide portion and a second polypeptide portion e.g. a second polypeptide portion having an amino acid sequence unrelated to . The second polypeptide portion can be e.g. any of glutathione 5 transferase a DNA binding domain or a polymerase activating domain. In preferred embodiment the fusion protein can be used in a two hybrid assay.

Polypeptides of the invention include those which arise as a result of alternative transcription events alternative RNA splicing events and alternative translational and postranslational events.

In a preferred embodiment the encoded polypeptide differs e.g. by amino acid substitution addition or deletion of at least one amino acid residue in amino acid sequence at 1 2 3 5 10 or more residues from a sequence of the invention contained in the Sequence Listing. The differences however are such that the encoded polypeptide exhibits a biological activity e.g. the encoded enzyme retains a biological activity of a naturally occurring

In preferred embodiments the encoded polypeptide includes all or a fragment of an amino acid sequence of the invention contained in the Sequence Listing fused in reading frame to additional amino acid residues preferably to residues encoded by genomic DNA 5 or 3 to the genomic DNA which encodes a sequence of the invention contained in the Sequence Listing.

The strain 14453 from which genomic sequences have been sequenced has been deposited on Jun. 26 1997 in the American Type Culture Collection 10801 University Blvd. Manassas Va. 20110 2209 and assigned the ATCC designation 55987.

Included in the invention are allelic variations natural mutants induced mutants proteins encoded by DNA that hybridize under high or low stringency conditions to a nucleic acid which encodes a polypeptide of the invention contained in the Sequence Listing for definitions of high and low stringency see Current Protocols in Molecular Biology John Wiley Sons New York 1989 6.3.1 6.3.6 hereby incorporated by reference and polypeptides specifically bound by antisera to polypeptides especially by antisera to an active site or binding domain of polypeptide. The invention also includes fragments preferably biologically active fragments. These and other polypeptides are also referred to herein as polypeptide analogs or variants.

The invention further provides nucleic acids e.g. RNA or DNA encoding a polypeptide of the invention. This includes double stranded nucleic acids as well as coding and antisense single strands.

In preferred embodiments the subject nucleic acid will include a transcriptional regulatory sequence e.g. at least one of a transcriptional promoter or transcriptional enhancer sequence operably linked to the gene sequence e.g. to render the gene sequence suitable for expression in a recombinant host cell.

In yet a further preferred embodiment the nucleic acid which encodes an polypeptide of the invention hybridizes under stringent conditions to a nucleic acid probe corresponding to at least 8 consecutive nucleotides of the invention contained in the Sequence Listing more preferably to at least 12 consecutive nucleotides of the invention contained in the Sequence Listing more preferably to at least 20 consecutive nucleotides of the invention contained in the Sequence Listing more preferably to at least 40 consecutive nucleotides of the invention contained in the Sequence Listing.

In another aspect the invention provides a substantially pure nucleic acid having a nucleotide sequence which encodes an polypeptide. In preferred embodiments the encoded polypeptide has biological activity the encoded polypeptide has an amino acid sequence at least 60 70 80 90 95 98 or 99 homologous to an amino acid sequence of the invention contained in the Sequence Listing the encoded polypeptide has an amino acid sequence essentially the same as an amino acid sequence of the invention contained in the Sequence Listing the encoded polypeptide is at least 5 10 20 50 100 or 150 amino acids in length the encoded polypeptide comprises at least 5 preferably at least 10 more preferably at least 20 more preferably at least 50 100 or 150 contiguous amino acids of the invention contained in the Sequence Listing.

In another aspect the invention encompasses a vector including a nucleic acid which encodes an polypeptide or an polypeptide variant as described herein a host cell transfected with the vector and a method of producing a recombinant polypeptide or polypeptide variant including culturing the cell e.g. in a cell culture medium and isolating an polypeptide or an polypeptide variant e.g. from the cell or from the cell culture medium.

In another series of embodiments the invention provides isolated nucleic acids comprising sequences at least about 8 nucleotides in length more preferably at least about 12 nucleotides in length and most preferably at least about 15 20 nucleotides in length that correspond to a subsequence of any one of SEQ ID NO 1 SEQ ID NO 2661 or complements thereof. Alternatively the nucleic acids comprise sequences contained within any ORF open reading frame including a complete protein coding sequence of which any of SEQ ID NO 1 SEQ ID NO 2661 forms a part. The invention encompasses sequence conservative variants and function conservative variants of these sequences. The nucleic acids may be DNA RNA DNA RNA duplexes protein nucleic acid PNA or derivatives thereof.

In another aspect the invention features a purified recombinant nucleic acid having at least 50 60 70 80 90 95 98 or 99 homology with a sequence of the invention contained in the Sequence Listing.

In another aspect the invention features nucleic acids capable of binding mRNA of . Such nucleic acid is capable of acting as antisense nucleic acid to control the translation of mRNA of . A further aspect features a nucleic acid which is capable of binding specifically to an nucleic acid. These nucleic acids are also referred to herein as complements and have utility as probes and as capture reagents.

In another aspect the invention features an expression system comprising an open reading frame corresponding to nucleic acid. The nucleic acid further comprises a control sequence compatible with an intended host. The expression system is useful for making polypeptides corresponding to nucleic acid.

In another aspect the invention features a cell transformed with the expression system to produce polypeptides.

In yet another embodiment the invention encompasses reagents for detecting bacterial infection including infection which comprise at least one derived nucleic acid defined by any one of SEQ ID NO 1 SEQ ID NO 2661 or sequence conservative or function conservative variants thereof. Alternatively the diagnostic reagents comprise polypeptide sequences that are contained within any open reading frames ORFs including complete protein coding sequences contained within any of SEQ ID NO 1 SEQ ID NO 2661 or polypeptide sequences contained within any of SEQ ID NO 2662 SEQ ID NO 5322 or polypeptides of which any of the above sequences forms a part or antibodies directed against any of the above peptide sequences or function conservative variants and or fragments thereof.

The invention further provides antibodies preferably monoclonal antibodies which specifically bind to the polypeptides of the invention. Methods are also provided for producing antibodies in a host animal. The methods of the invention comprise immunizing an animal with at least one derived immunogenic component wherein the immunogenic component comprises one or more of the polypeptides encoded by any one of SEQ ID NO 1 SEQ ID NO 2661 or sequence conservative or function conservative variants thereof or polypeptides that are contained within any ORFs including complete protein coding sequences of which any of SEQ ID NO 1 SEQ ID NO 2661 forms a part or polypeptide sequences contained within any of SEQ ID NO 2662 SEQ ID NO 5322 or polypeptides of which any of SEQ ID NO 2662 SEQ ID NO 5322 forms a part. Host animals include any warm blooded animal including without limitation mammals and birds. Such antibodies have utility as reagents for immunoassays to evaluate the abundance and distribution of specific antigens.

In yet another aspect the invention provides a method for detecting bacterial antigenic components in a sample which comprises the steps of i contacting a sample suspected to contain a bacterial antigenic component with a bacterial specific antibody under conditions in which a stable antigen antibody complex can form between the antibody and bacterial antigenic components in the sample and ii detecting any antigen antibody complex formed in step i wherein detection of an antigen antibody complex indicates the presence of at least one bacterial antigenic component in the sample. In different embodiments of this method the antibodies used are directed against a sequence encoded by any of SEQ ID NO 1 SEQ ID NO 2661 or sequence conservative or function conservative variants thereof or against a polypeptide sequence contained in any of SEQ ID NO 2662 SEQ ID NO 5322 or function conservative variants thereof.

In yet another aspect the invention provides a method for detecting antibacterial specific antibodies in a sample which comprises i contacting a sample suspected to contain antibacterial specific antibodies with a antigenic component under conditions in which a stable antigen antibody complex can form between the antigenic component and antibacterial antibodies in the sample and ii detecting any antigen antibody complex formed in step i wherein detection of an antigen antibody complex indicates the presence of antibacterial antibodies in the sample. In different embodiments of this method the antigenic component is encoded by a sequence contained in any of SEQ ID NO 1 SEQ ID NO 2661 or sequence conservative and function conservative variants thereof or is a polypeptide sequence contained in any of SEQ ID NO 2662 SEQ ID NO 5322 or function conservative variants thereof.

In another aspect the invention features a method of generating vaccines for immunizing an individual against . The method includes immunizing a subject with an polypeptide e.g. a surface or secreted polypeptide or active portion thereof and a pharmaceutically acceptable carrier. Such vaccines have therapeutic and prophylactic utilities.

In another aspect the invention features a method of evaluating a compound e.g. a polypeptide e.g. a fragment of a host cell polypeptide for the ability to bind an polypeptide. The method includes contacting the candidate compound with an polypeptide and determining if the compound binds or otherwise interacts with an polypeptide. Compounds which bind are candidates as activators or inhibitors of the bacterial life cycle. These assays can be performed in vitro or in vivo.

In another aspect the invention features a method of evaluating a compound e.g. a polypeptide e.g. a fragment of a host cell polypeptide for the ability to bind an nucleic acid e.g. DNA or RNA. The method includes contacting the candidate compound with an nucleic acid and determining if the compound binds or otherwise interacts with an polypeptide. Compounds which bind are candidates as activators or inhibitors of the bacterial life cycle. These assays can be performed in vitro or in vivo.

The sequences of the present invention include the specific nucleic acid and amino acid sequences set forth in the Sequence Listing that forms a part of the present specification and which are designated SEQ ID NO 1 SEQ ID NO 5322. Use of the terms SEQ ID NO 1 SEQ ID NO 2661 SEQ ID NO 2662 SEQ ID NO 5322 the sequences depicted in Table 2 etc. is intended for convenience to refer to each individual SEQ ID NO individually and is not intended to refer to the genus of these sequences. In other words it is a shorthand for listing all of these sequences individually. The invention encompasses each sequence individually as well as any combination thereof.

 Nucleic acid or polynucleotide as used herein refers to purine and pyrimidine containing polymers of any length either polyribonucleotides or polydeoxyribonucleotides or mixed polyribo polydeoxyribo nucleotides. This includes single and double stranded molecules i.e. DNA DNA DNA RNA and RNA RNA hybrids as well as protein nucleic acids PNA formed by conjugating bases to an amino acid backbone. This also includes nucleic acids containing modified bases.

A nucleic acid or polypeptide sequence that is derived from a designated sequence refers to a sequence that corresponds to a region of the designated sequence. For nucleic acid sequences this encompasses sequences that are homologous or complementary to the sequence as well as sequence conservative variants and function conservative variants. For polypeptide sequences this encompasses function conservative variants. Sequence conservative variants are those in which a change of one or more nucleotides in a given codon position results in no alteration in the amino acid encoded at that position. Function conservative variants are those in which a given amino acid residue in a polypeptide has been changed without altering the overall conformation and function of the native polypeptide including but not limited to replacement of an amino acid with one having similar physico chemical properties such as for example acidic basic hydrophobic and the like . Function conservative variants also include any polypeptides that have the ability to elicit antibodies specific to a designated polypeptide.

An derived nucleic acid or polypeptide sequence may or may not be present in other bacterial species and may or may not be present in all strains. This term is intended to refer to the source from which the sequence was originally isolated. Thus a derived polypeptide as used herein may be used e.g. as a target to screen for a broad spectrum antibacterial agent to search for homologous proteins in other species of bacteria or in eukaryotic organisms such as fungi and humans etc.

A purified or isolated polypeptide or a substantially pure preparation of a polypeptide are used interchangeably herein and as used herein mean a polypeptide that has been separated from other proteins lipids and nucleic acids with which it naturally occurs. Preferably the polypeptide is also separated from substances e.g. antibodies or gel matrix e.g. polyacrylamide which are used to purify it. Preferably the polypeptide constitutes at least 10 20 50 70 80 or 95 dry weight of the purified preparation. Preferably the preparation contains sufficient polypeptide to allow protein sequencing at least 1 10 or 100 mg of the polypeptide.

A purified preparation of cells refers to in the case of plant or animal cells an in vitro preparation of cells and not an entire intact plant or animal. In the case of cultured cells or microbial cells it consists of a preparation of at least 10 and more preferably 50 of the subject cells.

A purified or isolated or a substantially pure nucleic acid e.g. a substantially pure DNA are terms used interchangeably herein is a nucleic acid which is one or both of the following not immediately contiguous with both of the coding sequences with which it is immediately contiguous i.e. one at the 5 end and one at the 3 end in the naturally occurring genome of the organism from which the nucleic acid is derived or which is substantially free of a nucleic acid with which it occurs in the organism from which the nucleic acid is derived. The term includes for example a recombinant DNA which is incorporated into a vector e.g. into an autonomously replicating plasmid or virus or into the genomic DNA of a prokaryote or eukaryote or which exists as a separate molecule e.g. a cDNA or a genomic DNA fragment produced by PCR or restriction endonuclease treatment independent of other DNA sequences. Substantially pure DNA also includes a recombinant DNA which is part of a hybrid gene encoding additional DNA sequence.

A contig as used herein is a nucleic acid representing a continuous stretch of genomic sequence of an organism.

An open reading frame also referred to herein as ORF is a region of nucleic acid which encodes a polypeptide. This region usually represents the total coding region for the polypeptide and can be determined from a stop to stop codon or from a start to stop codon.

As used herein a coding sequence is a nucleic acid which is transcribed into messenger RNA and or translated into a polypeptide when placed under the control of appropriate regulatory sequences. The boundaries of the coding sequence are determined by a translation start codon at the five prime terminus and a translation stop codon at the three prime terminus. A coding sequence can include but is not limited to messenger RNA synthetic DNA and recombinant nucleic acid sequences.

A complement of a nucleic acid as used herein refers to an anti parallel or antisense sequence that participates in Watson Crick base pairing with the original sequence.

As used herein the term probe refers to a nucleic acid peptide or other chemical entity which specifically binds to a molecule of interest. Probes are often associated with or capable of associating with a label. A label is a chemical moiety capable of detection. Typical labels comprise dyes radioisotopes luminescent and chemiluminescent moieties fluorophores enzymes precipitating agents amplification sequences and the like. Similarly a nucleic acid peptide or other chemical entity which specifically binds to a molecule of interest and immobilizes such molecule is referred herein as a capture ligand . Capture ligands are typically associated with or capable of associating with a support such as nitro cellulose glass nylon membranes beads particles and the like. The specificity of hybridization is dependent on conditions such as the base pair composition of the nucleotides and the temperature and salt concentration of the reaction. These conditions are readily discernable to one of ordinary skill in the art using routine experimentation.

 Homologous refers to the sequence similarity or sequence identity between two polypeptides or between two nucleic acid molecules. When a position in both of the two compared sequences is occupied by the same base or amino acid monomer subunit e.g. if a position in each of two DNA molecules is occupied by adenine then the molecules are homologous at that position. The percent of homology between two sequences is a function of the number of matching or homologous positions shared by the two sequences divided by the number of positions compared 100. For example if 6 of 10 of the positions in two sequences are matched or homologous then the two sequences are 60 homologous. By way of example the DNA sequences ATTGCC and TATGGC share 50 homology. Generally a comparison is made when two sequences are aligned to give maximum homology.

Nucleic acids are hybridizable to each other when at least one strand of a nucleic acid can anneal to the other nucleic acid under defined stringency conditions. Stringency of hybridization is determined by a the temperature at which hybridization and or washing is performed and b the ionic strength and polarity of the hybridization and washing solutions. Hybridization requires that the two nucleic acids contain complementary sequences depending on the stringency of hybridization however mismatches may be tolerated. Typically hybridization of two sequences at high stringency such as for example in a solution of 0.5 SSC at 65 C. requires that the sequences be essentially completely homologous. Conditions of intermediate stringency such as for example 2 SSC at 65 C. and low stringency such as for example 2 SSC at 55 C. require correspondingly less overall complementarity between the hybridizing sequences. 1 SSC is 0.15 M NaCl 0.015 M Na citrate .

As used herein the term surface protein refers to all surface accessible proteins e.g. inner and outer membrane proteins proteins adhering to the cell wall and secreted proteins.

A polypeptide has biological activity if it has one two and preferably more of the following properties 1 if when expressed in the course of an infection it can promote or mediate the attachment of to a cell 2 it has an enzymatic activity structural or regulatory function characteristic of an protein 3 or the gene which encodes it can rescue a lethal mutation in an gene. A polypeptide has biological activity if it is an antagonist agonist or super agonist of a polypeptide having one of the above listed properties.

A biologically active fragment or analog is one having an in vivo or in vitro activity which is characteristic of the polypeptides of the invention contained in the Sequence Listing or of other naturally occurring polypeptides e.g. one or more of the biological activities described herein. Especially preferred are fragments which exist in vivo e.g. fragments which arise from post transcriptional processing or which arise from translation of alternatively spliced RNA s. Fragments include those expressed in native or endogenous cells as well as those made in expression systems e.g. in CHO cells. Because peptides such as polypeptides often exhibit a range of physiological properties and because such properties may be attributable to different portions of the molecule a useful fragment or analog is one which exhibits a biological activity in any biological assay for activity. Most preferably the fragment or analog possesses 10 preferably 40 more preferably 60 70 80 or 90 or greater of the activity of in any in vivo or in vitro assay.

Analogs can differ from naturally occurring polypeptides in amino acid sequence or in ways that do not involve sequence or both. Non sequence modifications include changes in acetylation methylation phosphorylation carboxylation or glycosylation. Preferred analogs include polypeptides or biologically active fragments thereof whose sequences differ from the wild type sequence by one or more conservative amino acid substitutions or by one or more non conservative amino acid substitutions deletions or insertions which do not substantially diminish the biological activity of the polypeptide. Conservative substitutions typically include the substitution of one amino acid for another with similar characteristics e.g. substitutions within the following groups valine glycine glycine alanine valine isoleucine leucine aspartic acid glutamic acid asparagine glutamine serine threonine lysine arginine and phenylalanine tyrosine. Other conservative substitutions can be made in view of the table below.

Other analogs within the invention are those with modifications which increase peptide stability such analogs may contain for example one or more non peptide bonds which replace the peptide bonds in the peptide sequence. Also included are analogs that include residues other than naturally occurring L amino acids e.g. D amino acids or non naturally occurring or synthetic amino acids e.g. or amino acids and cyclic analogs.

As used herein the term fragment as applied to an analog will ordinarily be at least about 20 residues more typically at least about 40 residues preferably at least about 60 residues in length. Fragments of polypeptides can be generated by methods known to those skilled in the art. The ability of a candidate fragment to exhibit a biological activity of polypeptide can be assessed by methods known to those skilled in the art as described herein. Also included are polypeptides containing residues that are not required for biological activity of the peptide or that result from alternative mRNA splicing or alternative protein processing events.

An immunogenic component as used herein is a moiety such as an polypeptide analog or fragment thereof that is capable of eliciting a humoral and or cellular immune response in a host animal.

An antigenic component as used herein is a moiety such as an polypeptide analog or fragment thereof that is capable of binding to a specific antibody with sufficiently high affinity to form a detectable antigen antibody complex.

The term antibody as used herein is intended to include fragments thereof which are specifically reactive with polypeptides.

As used herein the term cell specific promoter means a DNA sequence that serves as a promoter i.e. regulates expression of a selected DNA sequence operably linked to the promoter and which effects expression of the selected DNA sequence in specific cells of a tissue. The term also covers so called leaky promoters which regulate expression of a selected DNA primarily in one tissue but cause expression in other tissues as well.

Misexpression as used herein refers to a non wild type pattern of gene expression. It includes expression at non wild type levels i.e. over or under expression a pattern of expression that differs from wild type in terms of the time or stage at which the gene is expressed e.g. increased or decreased expression as compared with wild type at a predetermined developmental period or stage a pattern of expression that differs from wild type in terms of decreased expression as compared with wild type in a predetermined cell type or tissue type a pattern of expression that differs from wild type in terms of the splicing size amino acid sequence post translational modification or biological activity of the expressed polypeptide a pattern of expression that differs from wild type in terms of the effect of an environmental stimulus or extracellular stimulus on expression of the gene e.g. a pattern of increased or decreased expression as compared with wild type in the presence of an increase or decrease in the strength of the stimulus.

As used herein host cells and other such terms denoting microorganisms or higher eukaryotic cell lines cultured as unicellular entities refers to cells which can become or have been used as recipients for a recombinant vector or other transfer DNA and include the progeny of the original cell which has been transfected. It is understood by individuals skilled in the art that the progeny of a single parental cell may not necessarily be completely identical in genomic or total DNA compliment to the original parent due to accident or deliberate mutation.

As used herein the term control sequence refers to a nucleic acid having a base sequence which is recognized by the host organism to effect the expression of encoded sequences to which they are ligated. The nature of such control sequences differs depending upon the host organism in prokaryotes such control sequences generally include a promoter ribosomal binding site terminators and in some cases operators in eukaryotes generally such control sequences include promoters terminators and in some instances enhancers. The term control sequence is intended to include at a minimum all components whose presence is necessary for expression and may also include additional components whose presence is advantageous for example leader sequences.

As used herein the term operably linked refers to sequences joined or ligated to function in their intended manner. For example a control sequence is operably linked to coding sequence by ligation in such a way that expression of the coding sequence is achieved under conditions compatible with the control sequence and host cell.

The metabolism of a substance as used herein means any aspect of the expression function action or regulation of the substance. The metabolism of a substance includes modifications e.g. covalent or non covalent modifications of the substance. The metabolism of a substance includes modifications e.g. covalent or non covalent modification the substance induces in other substances. The metabolism of a substance also includes changes in the distribution of the substance. The metabolism of a substance includes changes the substance induces in the distribution of other substances.

A sample as used herein refers to a biological sample such as for example tissue or fluid isolated from an individual including without limitation plasma serum cerebrospinal fluid lymph tears saliva and tissue sections or from in vitro cell culture constituents as well as samples from the environment.

Technical and scientific terms used herein have the meanings commonly understood by one of ordinary skill in the art to which the present invention pertains unless otherwise defined. Reference is made herein to various methodologies known to those of skill in the art. Publications and other materials setting forth such known methodologies to which reference is made are incorporated herein by reference in their entireties as though set forth in full. The practice of the invention will employ unless otherwise indicated conventional techniques of chemistry molecular biology microbiology recombinant DNA and immunology which are within the skill of the art. Such techniques are explained fully in the literature. See e.g. Sambrook Fritsch and Maniatis 2nd ed. 1989 DNA Cloning Volumes I and II D. N Glover ed. 1985 M. J. Gait ed 1984 B. D. Hames S. J. Higgins eds. 1984 the series Academic Press Inc. particularly Vol. 154 and Vol. 155 Wu and Grossman eds. McPherson Quirke and Taylor eds. 1991 2d Edition 1989 Roitt et al. C. V. Mosby Company and New York 2d Edition 1991 Male et al. Grower Medical Publishing New York Volumes I and II 1985 D. N. Glover ed. 1984 M. L. Gait ed 1984 Hames and Higgins eds. 1986 R. I. Freshney ed. 1986 IRL Press Perbal 1984 and 1987 J. H. Miller and M. P. Calos eds. Cold Spring Harbor Laboratory .

Any suitable materials and or methods known to those of skill can be utilized in carrying out the present invention however preferred materials and or methods are described. Materials reagents and the like to which reference is made in the following description and examples are obtainable from commercial sources unless otherwise noted.

This invention provides nucleotide sequences of the genome of which thus comprises a DNA sequence library of genomic DNA. The detailed description that follows provides nucleotide sequences of and also describes how the sequences were obtained and how ORFs and protein coding sequences were identified. Also described are methods of using the disclosed sequences in methods including diagnostic and therapeutic applications. Furthermore the library can be used as a database for identification and comparison of medically important sequences in this and other strains of

To determine the genomic sequence of DNA was isolated from strain 14453 of and mechanically sheared by nebulization to a median size of 2 kb. Following size fractionation by gel electrophoresis the fragments were blunt ended ligated to adapter oligonucleotides and cloned into each of 20 different pMPX vectors Rice et al. abstracts of Meeting of Genome Mapping and Sequencing Cold Spring Harbor N.Y. 5 11 5 15 1994 p. 225 and the PUC19 vector to construct a series of shotgun subclone libraries.

DNA sequencing was achieved using two sequencing methods. The first method used multiplex sequencing procedures essentially as disclosed in Church et al. 1988240 185 U.S. Pat. Nos. 4 942 124 and 5 149 625 . DNA was extracted from pooled cultures and subjected to chemical or enzymatic sequencing. Sequencing reactions were resolved by electrophoresis and the products were transferred and covalently bound to nylon membranes. Finally the membranes were sequentially hybridized with a series of labelled oligonucleotides complimentary to tag sequences present in the different shotgun cloning vectors. In this manner a large number of sequences could be obtained from a single set of sequencing reactions. The remainder of the sequencing was performed on ABI377 automated DNA sequencers. The cloning and sequencing procedures are described in more detail in the Exemplification.

Individual sequence reads were assembled using PHRAP P. Green Abstracts of DOE Human Genome Program Contractor Grantee Workshop V January 1996 p. 157 . The average contig length was about 3 4 kb.

A variety of approaches are used to order the contigs so as to obtain a continuous sequence representing the entire genome. Synthetic oligonucleotides are designed that are complementary to sequences at the end of each contig. These oligonucleotides may be hybridized to libaries of genomic DNA in for example lambda phage vectors or plasmid vectors to identify clones that contain sequences corresponding to the junctional regions between individual contigs. Such clones are then used to isolate template DNA and the same oligonucleotides are used as primers in polymerase chain reaction PCR to amplify junctional fragments the nucleotide sequence of which is then determined.

The sequences were analyzed for the presence of open reading frames ORFs comprising at least 180 nucleotides. As a result of the initial analysis of ORFs based on stop to stop codon reads it should be understood that these ORFs may not correspond to the ORF of a naturally occurring polypeptide. These ORFs may contain start codons which indicate the initiation of protein synthesis of a naturally occurring polypeptide. Such start codons within the ORFs provided herein can be identified by those of ordinary skill in the relevant art and the resulting ORF and the encoded polypeptide is within the scope of this invention. For example within the ORFs a codon such as AUG or GUG encoding methionine or valine which is part of the initiation signal for protein synthesis can be identified and the portion of an ORF to corresponding to a naturally occurring polypeptide can be recognized.

The second analysis of the ORFs included identifying the start codons and the predicted coding regions. These ORFs provided in this invention were defined by one or more of the following methods evaluating the coding potential of such sequences with the program GENEMARK Borodovsky and McIninch 199317 123 distinguishing the coding from noncoding regions using the program Glimmer Fraser et al 1997 determining codon usage Staden et al. 10 141 and each predicted ORF amino acid sequence was compared with all protein sequences found in current GENBANK SWISS PROT and PIR databases using the BLAST algorithm. BLAST identifies local alignments occurring by chance between the ORF sequence and the sequence in the databank Altschal et al. 1990 L Mol. Biol. 215 403 410 . Homologous ORFs probabilities less than 10by chance and ORF s that are probably non homologous probabilities greater than 10by chance but have good codon usage were identified. Both homologous sequences and non homologous sequences with good codon usage are likely to encode proteins and are encompassed by the invention.

The nucleic acids of this invention may be obtained directly from the DNA of the above referenced strain by using the polymerase chain reaction PCR . See PCR A Practical Approach McPherson Quirke and Taylor eds. IRL Press Oxford UK 1991 for details about the PCR. High fidelity PCR can be used to ensure a faithful DNA copy prior to expression. In addition the authenticity of amplified products can be verified by conventional sequencing methods. Clones carrying the desired sequences described in this invention may also be obtained by screening the libraries by means of the PCR or by hybridization of synthetic oligonucleotide probes to filter lifts of the library colonies or plaques as known in the art see e.g. Sambrook et al. 2nd edition 1989 Cold Spring Harbor Press NY .

It is also possible to obtain nucleic acids encoding polypeptides from a cDNA library in accordance with protocols herein described. A cDNA encoding an polypeptide can be obtained by isolating total mRNA from an appropriate strain. Double stranded cDNAs can then be prepared from the total mRNA. Subsequently the cDNAs can be inserted into a suitable plasmid or viral e.g. bacteriophage vector using any one of a number of known techniques. Genes encoding polypeptides can also be cloned using established polymerase chain reaction techniques in accordance with the nucleotide sequence information provided by the invention. The nucleic acids of the invention can be DNA or RNA. Preferred nucleic acids of the invention are contained in the Sequence Listing.

The nucleic acids of the invention can also be chemically synthesized using standard techniques. Various methods of chemically synthesizing polydeoxynucleotides are known including solid phase synthesis which like peptide synthesis has been fully automated in commercially available DNA synthesizers See e.g. Itakura et al. U.S. Pat. No. 4 598 049 Caruthers et al. U.S. Pat. No. 4 458 066 and Itakura U.S. Pat. Nos. 4 401 796 and 4 373 071 incorporated by reference herein .

Nucleic acids isolated or synthesized in accordance with features of the present invention are useful by way of example without limitation as probes primers capture ligands antisense genes and for developing expression systems for the synthesis of proteins and peptides corresponding to such sequences. As probes primers capture ligands and antisense agents the nucleic acid normally consists of all or part approximately twenty or more nucleotides for specificity as well as the ability to form stable hybridization products of the nucleic acids of the invention contained in the Sequence Listing. These uses are described in further detail below.

A nucleic acid isolated or synthesized in accordance with the sequence of the invention contained in the Sequence Listing can be used as a probe to specifically detect . With the sequence information set forth in the present application sequences of twenty or more nucleotides are identified which provide the desired inclusivity and exclusivity with respect to and extraneous nucleic acids likely to be encountered during hybridization conditions. More preferably the sequence will comprise at least twenty to thirty nucleotides to convey stability to the hybridization product formed between the probe and the intended target molecules.

Sequences larger than 1000 nucleotides in length are difficult to synthesize but can be generated by recombinant DNA techniques. Individuals skilled in the art will readily recognize that the nucleic acids for use as probes can be provided with a label to facilitate detection of a hybridization product.

Nucleic acid isolated and synthesized in accordance with the sequence of the invention contained in the Sequence Listing can also be useful as probes to detect homologous regions especially homologous genes of other species using appropriate stringency hybridization conditions as described herein.

For use as a capture ligand the nucleic acid selected in the manner described above with respect to probes can be readily associated with a support. The manner in which nucleic acid is associated with supports is well known. Nucleic acid having twenty or more nucleotides in a sequence of the invention contained in the Sequence Listing have utility to separate nucleic acid from the nucleic acid of each other and other organisms. Nucleic acid having twenty or more nucleotides in a sequence of the invention contained in the Sequence Listing can also have utility to separate other species from each other and from other organisms. Preferably the sequence will comprise at least twenty nucleotides to convey stability to the hybridization product formed between the probe and the intended target molecules. Sequences larger than 1000 nucleotides in length are difficult to synthesize but can be generated by recombinant DNA techniques.

Nucleic acid isolated or synthesized in accordance with the sequences described herein have utility as primers for the amplification of nucleic acid. These nucleic acids may also have utility as primers for the amplification of nucleic acids in other species. With respect to polymerase chain reaction PCR techniques nucleic acid sequences of 10 15 nucleotides of the invention contained in the Sequence Listing have utility in conjunction with suitable enzymes and reagents to create copies of nucleic acid. More preferably the sequence will comprise twenty or more nucleotides to convey stability to the hybridization product formed between the primer and the intended target molecules. Binding conditions of primers greater than 100 nucleotides are more difficult to control to obtain specificity. High fidelity PCR can be used to ensure a faithful DNA copy prior to expression. In addition amplified products can be checked by conventional sequencing methods.

The copies can be used in diagnostic assays to detect specific sequences including genes from and or other species. The copies can also be incorporated into cloning and expression vectors to generate polypeptides corresponding to the nucleic acid synthesized by PCR as is described in greater detail herein.

Nucleic acid or nucleic acid hybridizing derivatives isolated or synthesized in accordance with the sequences described herein have utility as antisense agents to prevent the expression of genes. These sequences also have utility as antisense agents to prevent expression of genes of other species.

In one embodiment nucleic acid or derivatives corresponding to nucleic acids is loaded into a suitable carrier such as a liposome or bacteriophage for introduction into bacterial cells. For example a nucleic acid having twenty or more nucleotides is capable of binding to bacteria nucleic acid or bacteria messenger RNA. Preferably the antisense nucleic acid is comprised of 20 or more nucleotides to provide necessary stability of a hybridization product of non naturally occurring nucleic acid and bacterial nucleic acid and or bacterial messenger RNA. Nucleic acid having a sequence greater than 1000 nucleotides in length is difficult to synthesize but can be generated by recombinant DNA techniques. Methods for loading antisense nucleic acid in liposomes is known in the art as exemplified by U.S. Pat. No. 4 241 046 issued Dec. 23 1980 to Papahadjopoulos et al.

The present invention encompasses isolated polypeptides and nucleic acids derived from that are useful as reagents for diagnosis of bacterial infection components of effective antibacterial vaccines and or as targets for antibacterial drugs including anti drugs.

Table 2 provides a list of open reading frames ORFs in both strands. An ORF is a region of nucleic acid which encodes a polypeptide. This region normally represents a complete coding sequence or a total sequence and was determined from an initial analysis of stop to stop codons followed by the prediction of start codons. The first column lists the ORF designation. The second and third columns list the SEQ ID numbers for the nucleic acid and amino acid sequences corresponding to each ORF respectively. The fourth and fifth columns list the length of the nucleic acid ORF and the length of the amino acid ORF respectively. Most of the nucleotide sequences corresponding to each ORF begin at the first nucleotide of the start codon and end at the nucleotide immediately preceding the next downstream stop codon in the same reading frame. It will be recognized by one skilled in the art that the natural translation initiation sites will correspond to ATG GTG or TTG codons located within the ORFs. The natural initiation sites depend not only on the sequence of a start codon but also on the context of the DNA sequence adjacent to the start codon. Usually a recognizable ribosome binding site is found within 20 nucleotides upstream from the initiation codon. In some cases where genes are translationally coupled and coordinately expressed together in operons ribosome binding sites are not present but the initiation codon of a downstream gene may occur very close to or overlap the stop codon of the an upstream gene in the same operon. The correct start codons can be generally identified rapidly and efficiently because only a few codons need be tested. It is recognized that the translational machinery in bacteria initiates most polypeptide chains with the amino acid methionine. In some cases polypeptides are post translationally modified resulting in an N terminal amino acid other than methionine in vivo. The sixth and seventh columns provide metrics for assessing the likelihood of the homology match determined by the BLASTP2 algorithm as is known in the art to the genes indicated in the description field. Specifically the sixth column represents the Score for the match a higher score is a better match and the seventh column represents the P value for the match the probability that such a match could have occurred by chance the lower the value the more likely the match is valid . If a BLASTP2 score of less than 46 was obtained no value is reported in the table the P value . The description field provides where available the accession number AC or the Swissprot accession number SP the locus name LN Superfamily Classification CL the Organism OR Source of variant SR E.C. number EC the gene name GN the product name PN the Function Description FN the Map Position MP Left End LE Right End RE Coding Direction DI the Database from which the sequence originates DB and the description DE or notes NT for each ORF. This information allows one of ordinary skill in the art to determine a potential use and function for each identified coding sequence and as a result allows the use of the polypeptides of the present invention for commercial and industrial purposes.

Using the information provided in SEQ ID NO 1 SEQ ID NO 2661 and in Table 2 together with routine cloning and sequencing methods one of ordinary skill in the art will be able to clone and sequence all the nucleic acid fragments of interest including open reading frames ORFs encoding a large variety proteins of

Nucleic acid isolated or synthesized in accordance with the sequences described herein have utility to generate polypeptides. The nucleic acid of the invention exemplified in SEQ ID NO 1 SEQ ID NO 2661 and in Table 2 or fragments of said nucleic acid encoding active portions of polypeptides can be cloned into suitable vectors or used to isolate nucleic acid. The isolated nucleic acid is combined with suitable DNA linkers and cloned into a suitable vector.

The function of a specific gene or operon can be ascertained by expression in a bacterial strain under conditions where the activity of the gene product s specified by the gene or operon in question can be specifically measured. Alternatively a gene product may be produced in large quantities in an expressing strain for use as an antigen an industrial reagent for structural studies etc. This expression can be accomplished in a mutant strain which lacks the activity of the gene to be tested or in a strain that does not produce the same gene product s . This includes but is not limited to Eucaryotic species such as the yeast strains or other Archaea and Eubacteria such as or . In some cases the expression host will utilize the natural promoter whereas in others it will be necessary to drive the gene with a promoter sequence derived from the expressing organism e.g. an beta galactosidase promoter for expression in .

To express a gene product using the natural promoter a procedure such as the following can be used. A restriction fragment containing the gene of interest together with its associated natural promoter element and regulatory sequences identified using the DNA sequence data is cloned into an appropriate recombinant plasmid containing an origin of replication that functions in the host organism and an appropriate selectable marker. This can be accomplished by a number of procedures known to those skilled in the art. It is most preferably done by cutting the plasmid and the fragment to be cloned with the same restriction enzyme to produce compatible ends that can be ligated to join the two pieces together. The recombinant plasmid is introduced into the host organism by for example electroporation and cells containing the recombinant plasmid are identified by selection for the marker on the plasmid. Expression of the desired gene product is detected using an assay specific for that gene product.

In the case of a gene that requires a different promoter the body of the gene coding sequence is specifically excised and cloned into an appropriate expression plasmid. This subcloning can be done by several methods but is most easily accomplished by PCR amplification of a specific fragment and ligation into an expression plasmid after treating the PCR product with a restriction enzyme or exonuclease to create suitable ends for cloning.

A suitable host cell for expression of a gene can be any procaryotic or eucaryotic cell. For example an polypeptide can be expressed in bacterial cells such as or insect cells baculovirus yeast or mammalian cells such as Chinese hamster ovary cell CHO . Other suitable host cells are known to those skilled in the art.

Expression in eucaryotic cells such as mammalian yeast or insect cells can lead to partial or complete glycosylation and or formation of relevant inter or intra chain disulfide bonds of a recombinant peptide product. Examples of vectors for expression in yeast include pYepSec1 Baldari. et al. 1987 6 229 234 pMFa Kurjan and Herskowitz 1982 30 933 943 pJRY88 Schultz et al. 1987 54 113 123 and pYES2 Invitrogen Corporation San Diego Calif. . Baculovirus vectors available for expression of proteins in cultured insect cells SF 9 cells include the pAc series Smith et al. 1983 3 2156 2165 and the pVL series Lucklow V. A. and Summers M. D. 1989 170 31 39 . Generally COS cells Gluzman Y. 1981 Cell 23 175 182 are used in conjunction with such vectors as pCDM 8 Aruffo A. and Seed B. 1987 84 8573 8577 for transient amplification expression in mammalian cells while CHO dhfr Chinese Hamster Ovary cells are used with vectors such as pMT2PC Kaufman et al. 1987 6 187 195 for stable amplification expression in mammalian cells. Vector DNA can be introduced into mammalian cells via conventional techniques such as calcium phosphate or calcium chloride co precipitation DEAE dextran mediated transfection or electroporation. Suitable methods for transforming host cells can be found in Sambrook et al. 2nd Edition Cold Spring Harbor Laboratory press 1989 and other laboratory textbooks.

Expression in procaryotes is most often carried out in with either fusion or non fusion inducible expression vectors. Fusion vectors usually add a number of NHterminal amino acids to the expressed target gene. These NHterminal amino acids often are referred to as a reporter group or an affinity purification group. Such reporter groups usually serve two purposes 1 to increase the solubility of the target recombinant protein and 2 to aid in the purification of the target recombinant protein by acting as a ligand in affinity purification. Often in fusion expression vectors a proteolytic cleavage site is introduced at the junction of the reporter group and the target recombinant protein to enable separation of the target recombinant protein from the reporter group subsequent to purification of the fusion protein. Such enzymes and their cognate recognition sequences include Factor Xa thrombin and enterokinase. Typical fusion expression vectors include pGEX Amrad Corp. Melbourne Australia pMAL New England Biolabs Beverly Mass. and pRIT5 Pharmacia Piscataway N.J. which fuse glutathione S transferase maltose E binding protein or protein A respectively to the target recombinant protein. A preferred reporter group is poly His which may be fused to the amino or carboxy terminus of the protein and which renders the recombinant fusion protein easily purifiable by metal chelate chromatography.

Inducible non fusion expression vectors include pTrc Amann et al. 1988 Gene 69 301 315 and pET11d Studier et al. 185 Academic Press San Diego Calif. 1990 60 89 . While target gene expression relies on host RNA polymerase transcription from the hybrid trp lac fusion promoter in pTrc expression of target genes inserted into pET11d relies on transcription from the T7 gn10 lac 0 fusion promoter mediated by coexpressed viral RNA polymerase T7 gn1 . This viral polymerase is supplied by host strains BL21 DE3 or HMS174 DE3 from a resident prophage harboring a T7 gnl under the transcriptional control of the lacUV 5 promoter.

For example a host cell transfected with a nucleic acid vector directing expression of a nucleotide sequence encoding an polypeptide can be cultured under appropriate conditions to allow expression of the polypeptide to occur. The polypeptide may be secreted and isolated from a mixture of cells and medium containing the peptide. Alternatively the polypeptide may be retained cytoplasmically and the cells harvested lysed and the protein isolated. A cell culture includes host cells media and other byproducts. Suitable media for cell culture are well known in the art. Polypeptides of the invention can be isolated from cell culture medium host cells or both using techniques known in the art for purifying proteins including ion exchange chromatography gel filtration chromatography ultrafiltration electrophoresis and immunoaffinity purification with antibodies specific for such polypeptides. Additionally in many situations polypeptides can be produced by chemical cleavage of a native protein e.g. tryptic digestion and the cleavage products can then be purified by standard techniques.

In the case of membrane bound proteins these can be isolated from a host cell by contacting a membrane associated protein fraction with a detergent forming a solubilized complex where the membrane associated protein is no longer entirely embedded in the membrane fraction and is solubilized at least to an extent which allows it to be chromatographically isolated from the membrane fraction. Several different criteria are used for choosing a detergent suitable for solubilizing these complexes. For example one property considered is the ability of the detergent to solubilize the protein within the membrane fraction at minimal denaturation of the membrane associated protein allowing for the activity or functionality of the membrane associated protein to return upon reconstitution of the protein. Another property considered when selecting the detergent is the critical micelle concentration CMC of the detergent in that the detergent of choice preferably has a high CMC value allowing for ease of removal after reconstitution. A third property considered when selecting a detergent is the hydrophobicity of the detergent. Typically membrane associated proteins are very hydrophobic and therefore detergents which are also hydrophobic e.g. the triton series would be useful for solubilizing the hydrophobic proteins. Another property important to a detergent can be the capability of the detergent to remove the protein with minimal protein protein interaction facilitating further purification. A fifth property of the detergent which should be considered is the charge of the detergent. For example if it is desired to use ion exchange resins in the purification process then preferably detergent should be an uncharged detergent. Chromatographic techniques which can be used in the final purification step are known in the art and include hydrophobic interaction lectin affinity ion exchange dye affinity and immunoaffinity.

One strategy to maximize recombinant peptide expression in is to express the protein in a host bacteria with an impaired capacity to proteolytically cleave the recombinant protein Gottesman S. 185 Academic Press San Diego Calif. 1990 119 128 . Another strategy would be to alter the nucleic acid encoding an peptide to be inserted into an expression vector so that the individual codons for each amino acid would be those preferentially utilized in highly expressed proteins Wada et al. 1992 20 2111 2118 . Such alteration of nucleic acids of the invention can be carried out by standard DNA synthesis techniques.

The nucleic acids of the invention can also be chemically synthesized using standard techniques. Various methods of chemically synthesizing polydeoxynucleotides are known including solid phase synthesis which like peptide synthesis has been fully automated in commercially available DNA synthesizers See e.g. Itakura et al. U.S. Pat. No. 4 598 049 Caruthers et al. U.S. Pat. No. 4 458 066 and Itakura U.S. Pat. Nos. 4 401 796 and 4 373 071 incorporated by reference herein .

The present invention provides a library of derived nucleic acid sequences. The libraries provide probes primers and markers which can be used as markers in epidemiological studies. The present invention also provides a library of derived nucleic acid sequences which comprise or encode targets for therapeutic drugs.

Nucleic acids comprising any of the sequences disclosed herein or sub sequences thereof can be prepared by standard methods using the nucleic acid sequence information provided in SEQ ID NO 1 SEQ ID NO 2661. For example DNA can be chemically synthesized using e.g. the phosphoramidite solid support method of Matteucci et al. 1981103 3185 the method of Yoo et al. 1989764 17078 or other well known methods. This can be done by sequentially linking a series of oligonucleotide cassettes comprising pairs of synthetic oligonucleotides as described below.

Of course due to the degeneracy of the genetic code many different nucleotide sequences can encode polypeptides having the amino acid sequences defined by SEQ ID NO 2662 SEQ ID NO 5322 or sub sequences thereof. The codons can be selected for optimal expression in prokaryotic or eukaryotic systems. Such degenerate variants are also encompassed by this invention.

Insertion of nucleic acids typically DNAs encoding the polypeptides of the invention into a vector is easily accomplished when the termini of both the DNAs and the vector comprise compatible restriction sites. If this cannot be done it may be necessary to modify the termini of the DNAs and or vector by digesting back single stranded DNA overhangs generated by restriction endonuclease cleavage to produce blunt ends or to achieve the same result by filling in the single stranded termini with an appropriate DNA polymerase.

Alternatively any site desired may be produced e.g. by ligating nucleotide sequences linkers onto the termini. Such linkers may comprise specific oligonucleotide sequences that define desired restriction sites. Restriction sites can also be generated by the use of the polymerase chain reaction PCR . See e.g. Saiki et al. 1988239 48. The cleaved vector and the DNA fragments may also be modified if required by homopolymeric tailing.

In certain embodiments the invention encompasses isolated nucleic acid fragments comprising all or part of the individual nucleic acid sequences disclosed herein. The fragments are at least about 8 nucleotides in length preferably at least about 12 nucleotides in length and most preferably at least about 15 20 nucleotides in length.

The nucleic acids may be isolated directly from cells. Alternatively the polymerase chain reaction PCR method can be used to produce the nucleic acids of the invention using either chemically synthesized strands or genomic material as templates. Primers used for PCR can be synthesized using the sequence information provided herein and can further be designed to introduce appropriate new restriction sites if desirable to facilitate incorporation into a given vector for recombinant expression.

The nucleic acids of the present invention may be flanked by natural regulatory sequences or may be associated with heterologous sequences including promoters enhancers response elements signal sequences polyadenylation sequences introns 5 and 3 noncoding regions and the like. The nucleic acids may also be modified by many means known in the art. Non limiting examples of such modifications include methylation caps substitution of one or more of the naturally occurring nucleotides with an analog internucleotide modifications such as for example those with uncharged linkages e.g. methyl phosphonates phosphotriesters phosphoroamidates carbamates etc. and with charged linkages e.g. phosphorothioates phosphorodithioates etc. . Nucleic acids may contain one or more additional covalently linked moieties such as for example proteins e.g. nucleases toxins antibodies signal peptides poly L lysine etc. intercalators e.g. acridine psoralen etc. chelators e.g. metals radioactive metals iron oxidative metals etc. and alkylators. PNAs are also included. The nucleic acid may be derivatized by formation of a methyl or ethyl phosphotriester or an alkyl phosphoramidate linkage. Furthermore the nucleic acid sequences of the present invention may also be modified with a label capable of providing a detectable signal either directly or indirectly. Exemplary labels include radioisotopes fluorescent molecules biotin and the like.

The invention also provides nucleic acid vectors comprising the disclosed derived sequences or derivatives or fragments thereof. A large number of vectors including plasmid and fungal vectors have been described for replication and or expression in a variety of eukaryotic and prokaryotic hosts and may be used for gene therapy as well as for simple cloning or protein expression.

The encoded polypeptides may be expressed by using many known vectors such as pUC plasmids pET plasmids Novagen Inc. Madison Wis. or pRSET or pREP Invitrogen San Diego Calif. and many appropriate host cells using methods disclosed or cited herein or otherwise known to those skilled in the relevant art. The particular choice of vector host is not critical to the practice of the invention.

Recombinant cloning vectors will often include one or more replication systems for cloning or expression one or more markers for selection in the host e.g. antibiotic resistance and one or more expression cassettes. The inserted coding sequences may be synthesized by standard methods isolated from natural sources or prepared as hybrids etc. Ligation of the coding sequences to transcriptional regulatory elements and or to other amino acid coding sequences may be achieved by known methods. Suitable host cells may be transformed transfected infected as appropriate by any suitable method including electroporation CaClmediated DNA uptake fungal infection microinjection microprojectile or other established methods.

Appropriate host cells include bacteria archebacteria fungi especially yeast and plant and animal cells especially mammalian cells. Of particular interest are SF9 cells C129 cells 293 cells and CHO cells COS cells HeLa cells and immortalized mammalian myeloid and lymphoid cell lines. Preferred replication systems include M13 ColE1 SV40 baculovirus lambda adenovirus and the like. A large number of transcription initiation and termination regulatory regions have been isolated and shown to be effective in the transcription and translation of heterologous proteins in the various hosts. Examples of these regions methods of isolation manner of manipulation etc. are known in the art. Under appropriate expression conditions host cells can be used as a source of recombinantly produced derived peptides and polypeptides.

Advantageously vectors may also include a transcription regulatory element i.e. a promoter operably linked to the portion. The promoter may optionally contain operator portions and or ribosome binding sites. Non limiting examples of bacterial promoters compatible with include b lactamase penicillinase promoter lactose promoter tryptophan trp promoter araBAD arabinose operon promoter lambda derived Ppromoter and N gene ribosome binding site and the hybrid tac promoter derived from sequences of the trp and lac UV5 promoters. Non limiting examples of yeast promoters include 3 phosphoglycerate kinase promoter glyceraldehyde 3 phosphate dehydrogenase GAPDH promoter galactokinase GAL1 promoter galactoepimerase promoter and alcohol dehydrogenase ADH promoter. Suitable promoters for mammalian cells include without limitation viral promoters such as that from Simian Virus 40 SV40 Rous sarcoma virus RSV adenovirus ADV and bovine papilloma virus BPV . Mammalian cells may also require terminator sequences polyA addition sequences and enhancer sequences to increase expression. Sequences which cause amplification of the gene may also be desirable. Furthermore sequences that facilitate secretion of the recombinant product from cells including but not limited to bacteria yeast and animal cells such as secretory signal sequences and or prohormone pro region sequences may also be included. These sequences are well described in the art.

Nucleic acids encoding wild type or variant derived polypeptides may also be introduced into cells by recombination events. For example such a sequence can be introduced into a cell and thereby effect homologous recombination at the site of an endogenous gene or a sequence with substantial identity to the gene. Other recombination based methods such as nonhomologous recombinations or deletion of endogenous genes by homologous recombination may also be used.

The nucleic acids of the present invention find use as templates for the recombinant production of derived peptides or polypeptides.

The disclosed polypeptide and nucleic acid sequences or other sequences that are contained within ORFs including complete protein coding sequences of which any of the disclosed specific sequences forms a part are useful as target components for diagnosis and or treatment of caused infection

It will be understood that the sequence of an entire protein coding sequence of which each disclosed nucleic acid sequence forms a part can be isolated and identified based on each disclosed sequence. This can be achieved for example by using an isolated nucleic acid encoding the disclosed sequence or fragments thereof to prime a sequencing reaction with genomic DNA as template this is followed by sequencing the amplified product. The isolated nucleic acid encoding the disclosed sequence or fragments thereof can also be hybridized to genomic libraries to identify clones containing additional complete segments of the protein coding sequence of which the shorter sequence forms a part. Then the entire protein coding sequence or fragments thereof or nucleic acids encoding all or part of the sequence or sequence conservative or function conservative variants thereof may be employed in practicing the present invention.

Preferred sequences are those that are useful in diagnostic and or therapeutic applications. Diagnostic applications include without limitation nucleic acid based and antibody based methods for detecting bacterial infection. Therapeutic applications include without limitation vaccines passive immunotherapy and drug treatments directed against gene products that are both unique to bacteria and essential for growth and or replication of bacteria.

The disclosed genome sequence includes segments that direct the synthesis of ribonucleic acids and polypeptides as well as origins of replication promoters other types of regulatory sequences and intergenic nucleic acids. The invention encompasses nucleic acids encoding immunogenic components of vaccines and targets for agents effective against . Identification of said immunogenic components involved in the determination of the function of the disclosed sequences which can be achieved using a variety of approaches. Non limiting examples of these approaches are described briefly below.

Computer assisted comparison of the disclosed sequences with previously reported sequences present in publicly available databases is useful for identifying functional nucleic acid and polypeptide sequences. It will be understood that protein coding sequences for example may be compared as a whole and that a high degree of sequence homology between two proteins such as for example 80 90 at the amino acid level indicates that the two proteins also possess some degree of functional homology such as for example among enzymes involved in metabolism DNA synthesis or cell wall synthesis and proteins involved in transport cell division etc. In addition many structural features of particular protein classes have been identified and correlate with specific consensus sequences such as for example binding domains for nucleotides DNA metal ions and other small molecules sites for covalent modifications such as phosphorylation acylation and the like sites of protein protein interactions etc. These consensus sequences may be quite short and thus may represent only a fraction of the entire protein coding sequence. Identification of such a feature in an sequence is therefore useful in determining the function of the encoded protein and identifying useful targets of antibacterial drugs.

Of particular relevance to the present invention are structural features that are common to secretory transmembrane and surface proteins including secretion signal peptides and hydrophobic transmembrane domains. proteins identified as containing putative signal sequences and or transmembrane domains are useful as immunogenic components of vaccines.

Targets for therapeutic drugs according to the invention include but are not limited to polypeptides of the invention whether unique to or not that are essential for growth and or viability of under at least one growth condition. Polypeptides essential for growth and or viability can be determined by examining the effect of deleting and or disrupting the genes i.e. by so called gene knockout . Alternatively genetic footprinting can be used Smith et al. 199592 5479 6433 Published International Application WO 94 26933 U.S. Pat. No. 5 612 180 . Still other methods for assessing essentiality includes the ability to isolate conditional lethal mutations in the specific gene e.g. temperature sensitive mutations . Other useful targets for therapeutic drugs which include polypeptides that are not essential for growth or viability per se but lead to loss of viability of the cell can be used to target therapeutic agents to cells.

Because of the evolutionary relationship between different strains it is believed that the presently disclosed sequences are useful for identifying and or discriminating between previously known and new strains. It is believed that other strains will exhibit at least 70 sequence homology with the presently disclosed sequence. Systematic and routine analyses of DNA sequences derived from samples containing strains and comparison with the present sequence allows for the identification of sequences that can be used to discriminate between strains as well as those that are common to all strains. In one embodiment the invention provides nucleic acids including probes and peptide and polypeptide sequences that discriminate between different strains of . Strain specific components can also be identified functionally by their ability to elicit or react with antibodies that selectively recognize one or more strains.

In another embodiment the invention provides nucleic acids including probes and peptide and polypeptide sequences that are common to all strains but are not found in other bacterial species.

This invention encompasses isolated polypeptides encoded by the disclosed genomic sequences including the polypeptides of the invention contained in the Sequence Listing. Polypeptides of the invention are preferably at least 5 amino acid residues in length. Using the DNA sequence information provided herein the amino acid sequences of the polypeptides encompassed by the invention can be deduced using methods well known in the art. It will be understood that the sequence of an entire nucleic acid encoding an polypeptide can be isolated and identified based on an ORF that encodes only a fragment of the cognate protein coding region. This can be achieved for example by using the isolated nucleic acid encoding the ORF or fragments thereof to prime a polymerase chain reaction with genomic DNA as template this is followed by sequencing the amplified product.

The polypeptides of the present invention including function conservative variants of the disclosed ORFs may be isolated from wild type or mutant cells or from heterologous organisms or cells including but not limited to bacteria fungi insect plant and mammalian cells including into which a derived protein coding sequence has been introduced and expressed. Furthermore the polypeptides may be part of recombinant fusion proteins.

The alpha amino protecting groups are those known to be useful in the art of stepwise polypeptide synthesis. Included are acyl type protecting groups e.g. formyl trifluoroacetyl acetyl aromatic urethane type protecting groups e.g. benzyloxycarbonyl Cbz substituted benzyloxycarbonyl and 9 fluorenylmethyloxycarbonyl Fmoc aliphatic urethane protecting groups e.g. t butyloxycarbonyl Boc isopropyloxycarbonyl cyclohexyloxycarbonyl and alkyl type protecting groups e.g. benzyl triphenylmethyl. The preferred protecting group is Boc. The side chain protecting groups for Tyr include tetrahydropyranyl tert butyl trityl benzyl Cbz 4 Br Cbz and 2 6 dichlorobenzyl. The preferred side chain protecting group for Tyr is 2 6 dichlorobenzyl. The side chain protecting groups for Asp include benzyl 2 6 dichlorobenzyl methyl ethyl and cyclohexyl. The preferred side chain protecting group for Asp is cyclohexyl. The side chain protecting groups for Thr and Ser include acetyl benzoyl trityl tetrahydropyranyl benzyl 2 6 dichlorobenzyl and Cbz. The preferred protecting group for Thr and Ser is benzyl. The side chain protecting groups for Arg include nitro Tos Cbz adamantyloxycarbonyl and Boc. The preferred protecting group for Arg is Tos. The side chain amino group of Lys may be protected with Cbz 2 Cl Cbz Tos or Boc. The 2 Cl Cbz group is the preferred protecting group for Lys.

The side chain protecting groups selected must remain intact during coupling and not be removed during the deprotection of the amino terminus protecting group or during coupling conditions. The side chain protecting groups must also be removable upon the completion of synthesis using reaction conditions that will not alter the finished polypeptide.

Solid phase synthesis is usually carried out from the carboxy terminus by coupling the alpha amino protected side chain protected amino acid to a suitable solid support. An ester linkage is formed when the attachment is made to a chloromethyl or hydroxymethyl resin and the resulting polypeptide will have a free carboxyl group at the C terminus. Alternatively when a benzhydrylamine or p methylbenzhydrylamine resin is used an amide bond is formed and the resulting polypeptide will have a carboxamide group at the C terminus. These resins are commercially available and their preparation was described by Stewart et al. 1984 2nd Edition Pierce Chemical Co. Rockford Ill.

The C terminal amino acid protected at the side chain if necessary and at the alpha amino group is coupled to the benzhydrylamine resin using various activating agents including dicyclohexylcarbodiimide DCC N N diisopropyl carbodiimide and carbonyldiimidazole. Following the attachment to the resin support the alpha amino protecting group is removed using trifluoroacetic acid TFA or HCl in dioxane at a temperature between 0 and 25 C. Dimethylsulfide is added to the TFA after the introduction of methionine Met to suppress possible S alkylation. After removal of the alpha amino protecting group the remaining protected amino acids are coupled stepwise in the required order to obtain the desired sequence.

Various activating agents can be used for the coupling reactions including DCC N N diisopropyl carbodiimide benzotriazol 1 yl oxy tris dimethylamino phosphonium hexa fluorophosphate BOP and DCC hydroxybenzotriazole HOBt . Each protected amino acid is used in excess 2.0 equivalents and the couplings are usually carried out in N methylpyrrolidone NMP or in DMF CHClor mixtures thereof. The extent of completion of the coupling reaction is monitored at each stage e.g. by the ninhydrin reaction as described by Kaiser et al. 1970 Anal. Biochem. 34 595. In cases where incomplete coupling is found the coupling reaction is repeated. The coupling reactions can be performed automatically with commercially available instruments.

After the entire assembly of the desired polypeptide the polypeptide resin is cleaved with a reagent such as liquid HF for 1 2 hours at 0 C. which cleaves the polypeptide from the resin and removes all side chain protecting groups. A scavenger such as anisole is usually used with the liquid HF to prevent cations formed during the cleavage from alkylating the amino acid residues present in the polypeptide. The polypeptide resin may be deprotected with TFA dithioethane prior to cleavage if desired.

Side chain to side chain cyclization on the solid support requires the use of an orthogonal protection scheme which enables selective cleavage of the side chain functions of acidic amino acids e.g. Asp and the basic amino acids e.g. Lys . The 9 fluorenylmethyl Fm protecting group for the side chain of Asp and the 9 fluorenylmethyloxycarbonyl Fmoc protecting group for the side chain of Lys can be used for this purpose. In these cases the side chain protecting groups of the Boc protected polypeptide resin are selectively removed with piperidine in DMF. Cyclization is achieved on the solid support using various activating agents including DCC DCC HOBt or BOP. The HF reaction is carried out on the cyclized polypeptide resin as described above.

Methods for polypeptide purification are well known in the art including without limitation preparative disc gel electrophoresis isoelectric focusing HPLC reversed phase HPLC gel filtration ion exchange and partition chromatography and countercurrent distribution. For some purposes it is preferable to produce the polypeptide in a recombinant system in which the protein contains an additional sequence tag that facilitates purification such as but not limited to a polyhistidine sequence. The polypeptide can then be purified from a crude lysate of the host cell by chromatography on an appropriate solid phase matrix. Alternatively antibodies produced against a protein or against peptides derived therefrom can be used as purification reagents. Other purification methods are possible.

The present invention also encompasses derivatives and homologues of encoded polypeptides. For some purposes nucleic acid sequences encoding the peptides may be altered by substitutions additions or deletions that provide for functionally equivalent molecules i.e. function conservative variants. For example one or more amino acid residues within the sequence can be substituted by another amino acid of similar properties such as for example positively charged amino acids arginine lysine and histidine negatively charged amino acids aspartate and glutamate polar neutral amino acids and non polar amino acids. The isolated polypeptides may be modified by for example phosphorylation sulfation acylation or other protein modifications. They may also be modified with a label capable of providing a detectable signal either directly or indirectly including but not limited to radioisotopes and fluorescent compounds.

To identify derived polypeptides for use in the present invention essentially the complete genomic sequence of a virulent methicillin resistant isolate of isolate was analyzed. While in very rare instances a nucleic acid sequencing error may be revealed resolving a rare sequencing error is well within the art and such an occurrence will not prevent one skilled in the art from practicing the invention.

Also encompassed are any polypeptide sequences that are contained within the open reading frames ORFs including complete protein coding sequences of which any of SEQ ID NO 2662 SEQ ID NO 5322 forms a part. Table 2 which is appended herewith and which forms part of the present specification provides a putative identification of the particular function of a polypeptide which is encoded by each ORF. As a result one skilled in the art can use the polypeptides of the present invention for commercial and industrial purposes consistent with the type of putative identification of the polypeptide.

The present invention provides a library of derived polypeptide sequences and a corresponding library of nucleic acid sequences encoding the polypeptides wherein the polypeptides themselves or polypeptides contained within ORFs of which they form a part comprise sequences that are contemplated for use as components of vaccines. Non limiting examples of such sequences are listed by SEQ ID NO in Table 2 which is appended herewith and which forms part of the present specification.

The present invention also provides a library of derived polypeptide sequences and a corresponding library of nucleic acid sequences encoding the polypeptides wherein the polypeptides themselves or polypeptides contained within ORFs of which they form a part comprise sequences lacking homology to any known prokaryotic or eukaryotic sequences. Such libraries provide probes primers and markers which can be used to diagnose infection including use as markers in epidemiological studies. Non limiting examples of such sequences are listed by SEQ ID NO in Table 2 which is appended

The present invention also provides a library of derived polypeptide sequences and a corresponding library of nucleic acid sequences encoding the polypeptides wherein the polypeptides themselves or polypeptides contained within ORFs of which they form a part comprise targets for therapeutic drugs.

The selection of candidate protein antigens for vaccine development can be derived from the nucleic acids encoding polypeptides. First the ORF s can be analyzed for homology to other known exported or membrane proteins and analyzed using the discriminant analysis described by Klein et al. Klein P. Kanehsia M. and DeLisi C. 1985 815 468 476 for predicting exported and membrane proteins.

Homology searches can be performed using the BLAST algorithm contained in the Wisconsin Sequence Analysis Package Genetics Computer Group University Research Park 575 Science Drive Madison Wis. 53711 to compare each predicted ORF amino acid sequence with all sequences found in the current GenBank SWISS PROT and PIR databases. BLAST searches for local alignments between the ORF and the databank sequences and reports a probability score which indicates the probability of finding this sequence by chance in the database. ORF s with significant homology e.g. probabilities lower than 1 10that the homology is only due to random chance to membrane or exported proteins represent protein antigens for vaccine development. Possible functions can be provided to genes based on sequence homology to genes cloned in other organisms.

Discriminant analysis Klein et al. supra can be used to examine the ORF amino acid sequences. This algorithm uses the intrinsic information contained in the ORF amino acid sequence and compares it to information derived from the properties of known membrane and exported proteins. This comparison predicts which proteins will be exported membrane associated or cytoplasmic. ORF amino acid sequences identified as exported or membrane associated by this algorithm are likely protein antigens for vaccine development.

Based on the discovery of the gene products of the invention provided in the Sequence Listing one skilled in the art can alter the disclosed structure of genes e.g. by producing fragments or analogs and test the newly produced structures for activity. Examples of techniques known to those skilled in the relevant art which allow the production and testing of fragments and analogs are discussed below. These or analogous methods can be used to make and screen libraries of polypeptides e.g. libraries of random peptides or libraries of fragments or analogs of cellular proteins for the ability to bind polypeptides. Such screens are useful for the identification of inhibitors of

Fragments of a protein can be produced in several ways e.g. recombinantly by proteolytic digestion or by chemical synthesis. Internal or terminal fragments of a polypeptide can be generated by removing one or more nucleotides from one end for a terminal fragment or both ends for an internal fragment of a nucleic acid which encodes the polypeptide. Expression of the mutagenized DNA produces polypeptide fragments. Digestion with end nibbling endonucleases can thus generate DNA s which encode an array of fragments. DNA s which encode fragments of a protein can also be generated by random shearing restriction digestion or a combination of the above discussed methods.

Fragments can also be chemically synthesized using techniques known in the art such as conventional Merrifield solid phase f Moc or t Boc chemistry. For example peptides of the present invention may be arbitrarily divided into fragments of desired length with no overlap of the fragments or divided into overlapping fragments of a desired length.

Amino acid sequence variants of a protein can be prepared by random mutagenesis of DNA which encodes a protein or a particular domain or region of a protein. Useful methods include PCR mutagenesis and saturation mutagenesis. A library of random amino acid sequence variants can also be generated by the synthesis of a set of degenerate oligonucleotide sequences. Methods for screening proteins in a library of variants are elsewhere herein .

In PCR mutagenesis reduced Taq polymerase fidelity is used to introduce random mutations into a cloned fragment of DNA Leung et al. 19891 11 15 . The DNA region to be mutagenized is amplified using the polymerase chain reaction PCR under conditions that reduce the fidelity of DNA synthesis by Taq DNA polymerase e.g. by using a dGTP dATP ratio of five and adding Mn to the PCR reaction. The pool of amplified DNA fragments are inserted into appropriate cloning vectors to provide random mutant libraries.

Saturation mutagenesis allows for the rapid introduction of a large number of single base substitutions into cloned DNA fragments Mayers et al. 1985229 242 . This technique includes generation of mutations e.g. by chemical treatment or irradiation of single stranded DNA in vitro and synthesis of a complimentary DNA strand. The mutation frequency can be modulated by modulating the severity of the treatment and essentially all possible base substitutions can be obtained. Because this procedure does not involve a genetic selection for mutant fragments both neutral substitutions as well as those that alter function are obtained. The distribution of point mutations is not biased toward conserved sequence elements.

A library of homologs can also be generated from a set of degenerate oligonucleotide sequences. Chemical synthesis of a degenerate sequences can be carried out in an automatic DNA synthesizer and the synthetic genes then ligated into an appropriate expression vector. The synthesis of degenerate oligonucleotides is known in the art see for example Narang S A 1983 39 3 Itakura et al. 1981 3 ed. A G Walton Amsterdam Elsevier pp 273 289 Itakura et al. 1984 53 323 Itakura et al. 1984 198 1056 Ike et al. 1983 11 477. Such techniques have been employed in the directed evolution of other proteins see for example Scott et al. 1990 249 386 390 Roberts et al. 1992 89 2429 2433 Devlin et al. 1990 249 404 406 Cwirla et al. 1990 87 6378 6382 as well as U.S. Pat. Nos. 5 223 409 5 198 346 and 5 096 815 .

Non random or directed mutagenesis techniques can be used to provide specific sequences or mutations in specific regions. These techniques can be used to create variants which include e.g. deletions insertions or substitutions of residues of the known amino acid sequence of a protein. The sites for mutation can be modified individually or in series e.g. by 1 substituting first with conserved amino acids and then with more radical choices depending upon results achieved 2 deleting the target residue or 3 inserting residues of the same or a different class adjacent to the located site or combinations of options 1 3.

Alanine scanning mutagenesis is a useful method for identification of certain residues or regions of the desired protein that are preferred locations or domains for mutagenesis Cunningham and Wells 244 1081 1085 1989 . In alanine scanning a residue or group of target residues are identified e.g. charged residues such as Arg Asp His Lys and Glu and replaced by a neutral or negatively charged amino acid most preferably alanine or polyalanine . Replacement of an amino acid can affect the interaction of the amino acids with the surrounding aqueous environment in or outside the cell. Those domains demonstrating functional sensitivity to the substitutions are then refined by introducing further or other variants at or for the sites of substitution. Thus while the site for introducing an amino acid sequence variation is predetermined the nature of the mutation per se need not be predetermined. For example to optimize the performance of a mutation at a given site alanine scanning or random mutagenesis may be conducted at the target codon or region and the expressed desired protein subunit variants are screened for the optimal combination of desired activity.

Oligonucleotide mediated mutagenesis is a useful method for preparing substitution deletion and insertion variants of DNA see e.g. Adelman et al. 2 183 1983 . Briefly the desired DNA is altered by hybridizing an oligonucleotide encoding a mutation to a DNA template where the template is the single stranded form of a plasmid or bacteriophage containing the unaltered or native DNA sequence of the desired protein. After hybridization a DNA polymerase is used to synthesize an entire second complementary strand of the template that will thus incorporate the oligonucleotide primer and will code for the selected alteration in the desired protein DNA. Generally oligonucleotides of at least 25 nucleotides in length are used. An optimal oligonucleotide will have 12 to 15 nucleotides that are completely complementary to the template on either side of the nucleotide s coding for the mutation. This ensures that the oligonucleotide will hybridize properly to the single stranded DNA template molecule. The oligonucleotides are readily synthesized using techniques known in the art such as that described by Crea et al. . USA 75 5765 1978 .

Another method for preparing variants cassette mutagenesis is based on the technique described by Wells et al. 34 315 1985 . The starting material is a plasmid or other vector which includes the protein subunit DNA to be mutated. The codon s in the protein subunit DNA to be mutated are identified. There must be a unique restriction endonuclease site on each side of the identified mutation site s . If no such restriction sites exist they may be generated using the above described oligonucleotide mediated mutagenesis method to introduce them at appropriate locations in the desired protein subunit DNA. After the restriction sites have been introduced into the plasmid the plasmid is cut at these sites to linearize it. A double stranded oligonucleotide encoding the sequence of the DNA between the restriction sites but containing the desired mutation s is synthesized using standard procedures. The two strands are synthesized separately and then hybridized together using standard techniques. This double stranded oligonucleotide is referred to as the cassette. This cassette is designed to have 3 and 5 ends that are comparable with the ends of the linearized plasmid such that it can be directly ligated to the plasmid. This plasmid now contains the mutated desired protein subunit DNA sequence.

Combinatorial mutagenesis can also be used to generate mutants Ladner et al. WO 88 06630 . In this method the amino acid sequences for a group of homologs or other related proteins are aligned preferably to promote the highest homology possible. All of the amino acids which appear at a given position of the aligned sequences can be selected to create a degenerate set of combinatorial sequences. The variegated library of variants is generated by combinatorial mutagenesis at the nucleic acid level and is encoded by a variegated gene library. For example a mixture of synthetic oligonucleotides can be enzymatically ligated into gene sequences such that the degenerate set of potential sequences are expressible as individual peptides or alternatively as a set of larger fusion proteins containing the set of degenerate sequences.

It is possible to modify the structure of an polypeptide for such purposes as increasing solubility enhancing stability e.g. shelf life ex vivo and resistance to proteolytic degradation in vivo . A modified protein or peptide can be produced in which the amino acid sequence has been altered such as by amino acid substitution deletion or addition as described herein.

An peptide can also be modified by substitution of cysteine residues preferably with alanine serine threonine leucine or glutamic acid residues to minimize dimerization via disulfide linkages. In addition amino acid side chains of fragments of the protein of the invention can be chemically modified. Another modification is cyclization of the peptide.

In order to enhance stability and or reactivity an polypeptide can be modified to incorporate one or more polymorphisms in the amino acid sequence of the protein resulting from any natural allelic variation. Additionally D amino acids non natural amino acids or non amino acid analogs can be substituted or added to produce a modified protein within the scope of this invention. Furthermore an polypeptide can be modified using polyethylene glycol PEG according to the method of A. Sehon and co workers Wie et al. supra to produce a protein conjugated with PEG. In addition PEG can be added during chemical synthesis of the protein. Other modifications of proteins include reduction alkylation Tarr J. E. Silver ed. Humana Press Clifton N.J. 155 194 1986 acylation Tarr supra chemical coupling to an appropriate carrier Mishell and Shiigi eds WH Freeman San Francisco Calif. 1980 U.S. Pat. No. 4 939 239 or mild formalin treatment Marsh 1971 41 199 215 .

To facilitate purification and potentially increase solubility of an protein or peptide it is possible to add an amino acid fusion moiety to the peptide backbone. For example hexa histidine can be added to the protein for purification by immobilized metal ion affinity chromatography Hochuli E. et al. 1988 6 1321 1325 . In addition to facilitate isolation of peptides free of irrelevant sequences specific endoprotease cleavage sites can be introduced between the sequences of the fusion moiety and the peptide.

To potentially aid proper antigen processing of epitopes within an polypeptide canonical protease sensitive sites can be engineered between regions each comprising at least one epitope via recombinant or synthetic methods. For example charged amino acid pairs such as KK or RR can be introduced between regions within a protein or fragment during recombinant construction thereof. The resulting peptide can be rendered sensitive to cleavage by cathepsin and or other trypsin like enzymes which would generate portions of the protein containing one or more epitopes. In addition such charged amino acid residues can result in an increase in the solubility of the peptide.

Various techniques are known in the art for screening generated mutant gene products. Techniques for screening large gene libraries often include cloning the gene library into replicable expression vectors transforming appropriate cells with the resulting library of vectors and expressing the genes under conditions in which detection of a desired activity e.g. in this case binding to polypeptide or an interacting protein facilitates relatively easy isolation of the vector encoding the gene whose product was detected. Each of the techniques described below is amenable to high through put analysis for screening large numbers of sequences created e.g. by random mutagenesis techniques.

Two hybrid assays such as the system described above as with the other screening methods described herein can be used to identify polypeptides e.g. fragments or analogs of a naturally occurring polypeptide e.g. of cellular proteins or of randomly generated polypeptides which bind to an protein. The domain is used as the bait protein and the library of variants are expressed as prey fusion proteins. In an analogous fashion a two hybrid assay as with the other screening methods described herein can be used to find polypeptides which bind a polypeptide.

In one approach to screening assays the candidate peptides are displayed on the surface of a cell or viral particle and the ability of particular cells or viral particles to bind an appropriate receptor protein via the displayed product is detected in a panning assay . For example the gene library can be cloned into the gene for a surface membrane protein of a bacterial cell and the resulting fusion protein detected by panning Ladner et al. WO 88 06630 Fuchs et al. 1991 9 1370 1371 and Goward et al. 1992 18 136 140 . In a similar fashion a detectably labeled ligand can be used to score for potentially functional peptide homologs. Fluorescently labeled ligands e.g. receptors can be used to detect homologs which retain ligand binding activity. The use of fluorescently labeled ligands allows cells to be visually inspected and separated under a fluorescence microscope or where the morphology of the cell permits to be separated by a fluorescence activated cell sorter.

A gene library can be expressed as a fusion protein on the surface of a viral particle. For instance in the filamentous phage system foreign peptide sequences can be expressed on the surface of infectious phage thereby conferring two significant benefits. First since these phage can be applied to affinity matrices at concentrations well over 10phage per milliliter a large number of phage can be screened at one time. Second since each infectious phage displays a gene product on its surface if a particular phage is recovered from an affinity matrix in low yield the phage can be amplified by another round of infection. The group of almost identical filamentous phages M13 fd. and f1 are most often used in phage display libraries. Either of the phage gIII or gVIII coat proteins can be used to generate fusion proteins without disrupting the ultimate packaging of the viral particle. Foreign epitopes can be expressed at the NH terminal end of pIII and phage bearing such epitopes recovered from a large excess of phage lacking this epitope Ladner et al. PCT publication WO 90 02909 Garrard et al. PCT publication WO 92 09690 Marks et al. 1992 267 16007 16010 Griffiths et al. 1993 12 725 734 Clackson et al. 1991 352 624 628 and Barbas et al. 1992 89 4457 4461 .

A common approach uses the maltose receptor of the outer membrane protein LamB as a peptide fusion partner Charbit et al. 1986 5 3029 3037 . Oligonucleotides have been inserted into plasmids encoding the LamB gene to produce peptides fused into one of the extracellular loops of the protein. These peptides are available for binding to ligands e.g. to antibodies and can elicit an immune response when the cells are administered to animals. Other cell surface proteins e.g. OmpA Schorr et al. 1991 91 pp. 387 392 PhoE Agterberg et al. 1990 88 37 45 and PAL Fuchs et al. 1991 9 1369 1372 as well as large bacterial surface structures have served as vehicles for peptide display. Peptides can be fused to pilin a protein which polymerizes to form the pilus a conduit for interbacterial exchange of genetic information Thiry et al. 1989 55 984 993 . Because of its role in interacting with other cells the pilus provides a useful support for the presentation of peptides to the extracellular environment. Another large surface structure used for peptide display is the bacterial motive organ the flagellum. Fusion of peptides to the subunit protein flagellin offers a dense array of many peptide copies on the host cells Kuwajima et al. 1988 6 1080 1083 . Surface proteins of other bacterial species have also served as peptide fusion partners. Examples include the protein A and the outer membrane IgA protease of Hansson et al. 1992 174 4239 4245 and Klauser et al. 1990 9 1991 1999 .

In the filamentous phage systems and the LamB system described above the physical link between the peptide and its encoding DNA occurs by the containment of the DNA within a particle cell or phage that carries the peptide on its surface. Capturing the peptide captures the particle and the DNA within. An alternative scheme uses the DNA binding protein LacI to form a link between peptide and DNA Cull et al. 1992 89 1865 1869 . This system uses a plasmid containing the LacI gene with an oligonucleotide cloning site at its 3 end. Under the controlled induction by arabinose a LacI peptide fusion protein is produced. This fusion retains the natural ability of LacI to bind to a short DNA sequence known as LacO operator LacO . By installing two copies of LacO on the expression plasmid the LacI peptide fusion binds tightly to the plasmid that encoded it. Because the plasmids in each cell contain only a single oligonucleotide sequence and each cell expresses only a single peptide sequence the peptides become specifically and stablely associated with the DNA sequence that directed its synthesis. The cells of the library are gently lysed and the peptide DNA complexes are exposed to a matrix of immobilized receptor to recover the complexes containing active peptides. The associated plasmid DNA is then reintroduced into cells for amplification and DNA sequencing to determine the identity of the peptide ligands. As a demonstration of the practical utility of the method a large random library of dodecapeptides was made and selected on a monoclonal antibody raised against the opioid peptide dynorphin B. A cohort of peptides was recovered all related by a consensus sequence corresponding to a six residue portion of dynorphin B. Cull et al. 1992 89 1869 

This scheme sometimes referred to as peptides on plasmids differs in two important ways from the phage display methods. First the peptides are attached to the C terminus of the fusion protein resulting in the display of the library members as peptides having free carboxy termini. Both of the filamentous phage coat proteins pIII and pVIII are anchored to the phage through their C termini and the guest peptides are placed into the outward extending N terminal domains. In some designs the phage displayed peptides are presented right at the amino terminus of the fusion protein. Cwirla et al. 1990 87 6378 6382 A second difference is the set of biological biases affecting the population of peptides actually present in the libraries. The LacI fusion molecules are confined to the cytoplasm of the host cells. The phage coat fusions are exposed briefly to the cytoplasm during translation but are rapidly secreted through the inner membrane into the periplasmic compartment remaining anchored in the membrane by their C terminal hydrophobic domains with the N termini containing the peptides protruding into the periplasm while awaiting assembly into phage particles. The peptides in the LacI and phage libraries may differ significantly as a result of their exposure to different proteolytic activities. The phage coat proteins require transport across the inner membrane and signal peptidase processing as a prelude to incorporation into phage. Certain peptides exert a deleterious effect on these processes and are underrepresented in the libraries Gallop et al. 1994 37 9 1233 1251 . These particular biases are not a factor in the LacI display system.

The number of small peptides available in recombinant random libraries is enormous. Libraries of 10 10independent clones are routinely prepared. Libraries as large as 10recombinants have been created but this size approaches the practical limit for clone libraries. This limitation in library size occurs at the step of transforming the DNA containing randomized segments into the host bacterial cells. To circumvent this limitation an in vitro system based on the display of nascent peptides in polysome complexes has recently been developed. This display library method has the potential of producing libraries 3 6 orders of magnitude larger than the currently available phage phagemid or plasmid libraries. Furthermore the construction of the libraries expression of the peptides and screening is done in an entirely cell free format.

In one application of this method Gallop et al. 1994 37 9 1233 1251 a molecular DNA library encoding 1012 decapeptides was constructed and the library expressed in an S30 in vitro coupled transcription translation system. Conditions were chosen to stall the ribosomes on the mRNA causing the accumulation of a substantial proportion of the RNA in polysomes and yielding complexes containing nascent peptides still linked to their encoding RNA. The polysomes are sufficiently robust to be affinity purified on immobilized receptors in much the same way as the more conventional recombinant peptide display libraries are screened. RNA from the bound complexes is recovered converted to cDNA and amplified by PCR to produce a template for the next round of synthesis and screening. The polysome display method can be coupled to the phage display system. Following several rounds of screening cDNA from the enriched pool of polysomes was cloned into a phagemid vector. This vector serves as both a peptide expression vector displaying peptides fused to the coat proteins and as a DNA sequencing vector for peptide identification. By expressing the polysome derived peptides on phage one can either continue the affinity selection procedure in this format or assay the peptides on individual clones for binding activity in a phage ELISA or for binding specificity in a completion phage ELISA Barret et al. 1992 204 357 364 . To identify the sequences of the active peptides one sequences the DNA produced by the phagemid host.

The high through put assays described above can be followed by secondary screens in order to identify further biological activities which will e.g. allow one skilled in the art to differentiate agonists from antagonists. The type of a secondary screen used will depend on the desired activity that needs to be tested. For example an assay can be developed in which the ability to inhibit an interaction between a protein of interest and its respective ligand can be used to identify antagonists from a group of peptide fragments isolated though one of the primary screens described above.

Therefore methods for generating fragments and analogs and testing them for activity are known in the art. Once the core sequence of interest is identified it is routine for one skilled in the art to obtain analogs and fragments.

The invention also provides for reduction of the protein binding domains of the subject polypeptides to generate mimetics e.g. peptide or non peptide agents. The peptide mimetics are able to disrupt binding of a polypeptide to its counter ligand e.g. in the case of an polypeptide binding to a naturally occurring ligand. The critical residues of a subject polypeptide which are involved in molecular recognition of a polypeptide can be determined and used to generate derived peptidomimetics which competitively or noncompetitively inhibit binding of the polypeptide with an interacting polypeptide see for example European patent applications EP 412 762A and EP B31 080A .

For example scanning mutagenesis can be used to map the amino acid residues of a particular polypeptide involved in binding an interacting polypeptide peptidomimetic compounds e.g. diazepine or isoquinoline derivatives can be generated which mimic those residues in binding to an interacting polypeptide and which therefore can inhibit binding of an polypeptide to an interacting polypeptide and thereby interfere with the function of polypeptide. For instance non hydrolyzable peptide analogs of such residues can be generated using benzodiazepine e.g. see Freidinger et al. in G. R. Marshall ed. ESCOM Publisher Leiden Netherlands 1988 azepine e.g. see Huffman et al. in G. R. Marshall ed. ESCOM Publisher Leiden Netherlands 1988 substituted gama lactam rings Garvey et al. in G. R. Marshall ed. ESCOM Publisher Leiden Netherlands 1988 keto methylene pseudopeptides Ewenson et al. 1986 29 295 and Ewenson et al. in Proceedings of the 9th American Peptide Symposium Pierce Chemical Co. Rockland Ill. 1985 b turn dipeptide cores Nagai et al. 1985 26 647 and Sato et al. 1986 1 1231 and b aminoalcohols Gordon et al. 1985 126 419 and et al. 1986 134 71 .

This invention also features vaccine compositions for protection against infection by or for treatment of infection a gram negative spiral microaerophilic bacterium. In one embodiment the vaccine compositions contain one or more immunogenic components such as a surface protein from or portion thereof and a pharmaceutically acceptable carrier. Nucleic acids within the scope of the invention are exemplified by the nucleic acids of the invention contained in the Sequence Listing which encode surface proteins. Any nucleic acid encoding an immunogenic protein or portion thereof which is capable of expression in a cell can be used in the present invention. These vaccines have therapeutic and prophylactic utilities.

One aspect of the invention provides a vaccine composition for protection against infection by which contains at least one immunogenic fragment of an protein and a pharmaceutically acceptable carrier. Preferred fragments include peptides of at least about 10 amino acid residues in length preferably about 10 20 amino acid residues in length and more preferably about 12 16 amino acid residues in length.

Immunogenic components of the invention can be obtained for example by screening polypeptides recombinantly produced from the corresponding fragment of the nucleic acid encoding the full length protein. In addition fragments can be chemically synthesized using techniques known in the art such as conventional Merrifield solid phase f Moc or t Boc chemistry.

In one embodiment immunogenic components are identified by the ability of the peptide to stimulate T cells. Peptides which stimulate T cells as determined by for example T cell proliferation or cytokine secretion are defined herein as comprising at least one T cell epitope. T cell epitopes are believed to be involved in initiation and perpetuation of the immune response to the protein allergen which is responsible for the clinical symptoms of allergy. These T cell epitopes are thought to trigger early events at the level of the T helper cell by binding to an appropriate HLA molecule on the surface of an antigen presenting cell thereby stimulating the T cell subpopulation with the relevant T cell receptor for the epitope. These events lead to T cell proliferation lymphokine secretion local inflammatory reactions recruitment of additional immune cells to the site of antigen T cell interaction and activation of the B cell cascade leading to the production of antibodies. A T cell epitope is the basic element or smallest unit of recognition by a T cell receptor where the epitope comprises amino acids essential to receptor recognition e.g. approximately 6 or 7 amino acid residues . Amino acid sequences which mimic those of the T cell epitopes are within the scope of this invention.

Screening immunogenic components can be accomplished using one or more of several different assays. For example in vitro peptide T cell stimulatory activity is assayed by contacting a peptide known or suspected of being immunogenic with an antigen presenting cell which presents appropriate MHC molecules in a T cell culture. Presentation of an immunogenic peptide in association with appropriate MHC molecules to T cells in conjunction with the necessary co stimulation has the effect of transmitting a signal to the T cell that induces the production of increased levels of cytokines particularly of interleukin 2 and interleukin 4. The culture supernatant can be obtained and assayed for interleukin 2 or other known cytokines. For example any one of several conventional assays for interleukin 2 can be employed such as the assay described in 86 1333 1989 the pertinent portions of which are incorporated herein by reference. A kit for an assay for the production of interferon is also available from Genzyme Corporation Cambridge Mass. .

Alternatively a common assay for T cell proliferation entails measuring tritiated thymidine incorporation. The proliferation of T cells can be measured in vitro by determining the amount of H labeled thymidine incorporated into the replicating DNA of cultured cells. Therefore the rate of DNA synthesis and in turn the rate of cell division can be quantified.

Vaccine compositions of the invention containing immunogenic components e.g. polypeptide or fragment thereof or nucleic acid encoding an polypeptide or fragment thereof preferably include a pharmaceutically acceptable carrier. The term pharmaceutically acceptable carrier refers to a carrier that does not cause an allergic reaction or other untoward effect in patients to whom it is administered. Suitable pharmaceutically acceptable carriers include for example one or more of water saline phosphate buffered saline dextrose glycerol ethanol and the like as well as combinations thereof. Pharmaceutically acceptable carriers may further comprise minor amounts of auxiliary substances such as wetting or emulsifying agents preservatives or buffers which enhance the shelf life or effectiveness of the antibody. For vaccines of the invention containing polypeptides the polypeptide is co administered with a suitable adjuvant.

It will be apparent to those of skill in the art that the therapeutically effective amount of DNA or protein of this invention will depend inter alia upon the administration schedule the unit dose of antibody administered whether the protein or DNA is administered in combination with other therapeutic agents the immune status and health of the patient and the therapeutic activity of the particular protein or DNA.

Vaccine compositions are conventionally administered parenterally e.g. by injection either subcutaneously or intramuscularly. Methods for intramuscular immunization are described by Wolff et al. 1990 247 1465 1468 and by Sedegah et al. 1994 91 9866 9870. Other modes of administration include oral and pulmonary formulations suppositories and transdermal applications. Oral immunization is preferred over parenteral methods for inducing protection against infection by . Cain et. al. 1993 11 637 642. Oral formulations include such normally employed excipients as for example pharmaceutical grades of mannitol lactose starch magnesium stearate sodium saccharine cellulose magnesium carbonate and the like.

The vaccine compositions of the invention can include an adjuvant including but not limited to aluminum hydroxide N acetyl muramyl L threonyl D isoglutamine thr MDP N acetyl nor muramyl L alanyl D isoglutamine CGP 11637 referred to as nor MDP N acetylmuramyl L alanyl D isoglutaminyl L alanine 2 1 2 dipalmitoyl sn glycero 3 hydroxyphos phoryloxy ethylamine CGP 19835A referred to a MTP PE RIBI which contains three components from bacteria monophosphoryl lipid A trehalose dimycoloate cell wall skeleton MPL TDM CWS in a 2 squalene Tween 80 emulsion and cholera toxin. Others which may be used are non toxic derivatives of cholera toxin including its B subunit and or conjugates or genetically engineered fusions of the polypeptide with cholera toxin or its B subunit procholeragenoid fungal polysaccharides including schizophyllan muramyl dipeptide muramyl dipeptide derivatives phorbol esters labile toxin of non bacterial lysates block polymers or saponins.

Other suitable delivery methods include biodegradable microcapsules or immuno stimulating complexes ISCOMs cochleates or liposomes genetically engineered attenuated live vectors such as viruses or bacteria and recombinant chimeric virus like particles e.g. bluetongue. The amount of adjuvant employed will depend on the type of adjuvant used. For example when the mucosal adjuvant is cholera toxin it is suitably used in an amount of 5 mg to 50 mg for example 10 mg to 35 mg. When used in the form of microcapsules the amount used will depend on the amount employed in the matrix of the microcapsule to achieve the desired dosage. The determination of this amount is within the skill of a person of ordinary skill in the art.

Carrier systems in humans may include enteric release capsules protecting the antigen from the acidic environment of the stomach and including polypeptide in an insoluble form as fusion proteins. Suitable carriers for the vaccines of the invention are enteric coated capsules and polylactide glycolide microspheres. Suitable diluents are 0.2 N NaHCO3 and or saline.

Vaccines of the invention can be administered as a primary prophylactic agent in adults or in children as a secondary prevention after successful eradication of in an infected host or as a therapeutic agent in the aim to induce an immune response in a susceptible host to prevent infection by . The vaccines of the invention are administered in amounts readily determined by persons of ordinary skill in the art. Thus for adults a suitable dosage will be in the range of 10 mg to 10 g preferably 10 mg to 100 mg. A suitable dosage for adults will also be in the range of 5 mg to 500 mg. Similar dosage ranges will be applicable for children. Those skilled in the art will recognize that the optimal dose may be more or less depending upon the patient s body weight disease the route of administration and other factors. Those skilled in the art will also recognize that appropriate dosage levels can be obtained based on results with known oral vaccines such as for example a vaccine based on an lysate 6 mg dose daily up to total of 540 mg and with an enterotoxigenic purified antigen 4 doses of 1 mg Schulman et al. 150 917 921 1993 Boedecker et al. 999 A 222 1993 . The number of doses will depend upon the disease the formulation and efficacy data from clinical trials. Without intending any limitation as to the course of treatment the treatment can be administered over 3 to 8 doses for a primary immunization schedule over 1 month Boedeker 888 A 222 1993 .

In a preferred embodiment a vaccine composition of the invention can be based on a killed whole preparation with an immunogenic fragment of an protein of the invention expressed on its surface or it can be based on an lysate wherein the killed acts as a carrier or an adjuvant.

It will be apparent to those skilled in the art that some of the vaccine compositions of the invention are useful only for preventing infection some are useful only for treating infection and some are useful for both preventing and treating infection. In a preferred embodiment the vaccine composition of the invention provides protection against infection by stimulating humoral and or cell mediated immunity against . It should be understood that amelioration of any of the symptoms of infection is a desirable clinical goal including a lessening of the dosage of medication used to treat caused disease or an increase in the production of antibodies in the serum or mucous of patients.

The invention also includes antibodies specifically reactive with the subject polypeptide. Anti protein anti peptide antisera or monoclonal antibodies can be made by standard protocols See for example ed. by Harlow and Lane Cold Spring Harbor Press 1988 . A mammal such as a mouse a hamster or rabbit can be immunized with an immunogenic form of the peptide. Techniques for conferring immunogenicity on a protein or peptide include conjugation to carriers or other techniques well known in the art. An immunogenic portion of the subject polypeptide can be administered in the presence of adjuvant. The progress of immunization can be monitored by detection of antibody titers in plasma or serum. Standard ELISA or other immunoassays can be used with the immunogen as antigen to assess the levels of antibodies.

In a preferred embodiment the subject antibodies are immunospecific for antigenic determinants of the polypeptides of the invention e.g. antigenic determinants of a polypeptide of the invention contained in the Sequence Listing or a closely related human or non human mammalian homolog e.g. 90 homologous more preferably at least 95 homologous . In yet a further preferred embodiment of the invention the anti antibodies do not substantially cross react i.e. react specifically with a protein which is for example less than 80 percent homologous to a sequence of the invention contained in the Sequence Listing. By not substantially cross react it is meant that the antibody has a binding affinity for a non homologous protein which is less than 10 percent more preferably less than 5 percent and even more preferably less than 1 percent of the binding affinity for a protein of the invention contained in the Sequence Listing. In a most preferred embodiment there is no cross reactivity between bacterial and mammalian antigens.

The term antibody as used herein is intended to include fragments thereof which are also specifically reactive with polypeptides. Antibodies can be fragmented using conventional techniques and the fragments screened for utility in the same manner as described above for whole antibodies. For example F ab fragments can be generated by treating antibody with pepsin. The resulting F ab fragment can be treated to reduce disulfide bridges to produce Fab fragments. The antibody of the invention is further intended to include bispecific and chimeric molecules having an anti portion.

Both monoclonal and polyclonal antibodies Ab directed against polypeptides or polypeptide variants and antibody fragments such as Fab and F ab can be used to block the action of polypeptide and allow the study of the role of a particular polypeptide of the invention in aberrant or unwanted intracellular signaling as well as the normal cellular function of the and by microinjection of anti polypeptide antibodies of the present invention.

Antibodies which specifically bind epitopes can also be used in immunohistochemical staining of tissue samples in order to evaluate the abundance and pattern of expression of antigens. Anti polypeptide antibodies can be used diagnostically in immuno precipitation and immuno blotting to detect and evaluate levels in tissue or bodily fluid as part of a clinical testing procedure. Likewise the ability to monitor polypeptide levels in an individual can allow determination of the efficacy of a given treatment regimen for an individual afflicted with such a disorder. The level of an polypeptide can be measured in cells found in bodily fluid such as in urine samples or can be measured in tissue such as produced by gastric biopsy. Diagnostic assays using anti antibodies can include for example immunoassays designed to aid in early diagnosis of infections. The present invention can also be used as a method of detecting antibodies contained in samples from individuals infected by this bacterium using specific antigens.

Another application of anti polypeptide antibodies of the invention is in the immunological screening of cDNA libraries constructed in expression vectors such as lgt11 lgt18 23 IZAP and lORF8. Messenger libraries of this type having coding sequences inserted in the correct reading frame and orientation can produce fusion proteins. For instance lgt11 will produce fusion proteins whose amino termini consist of galactosidase amino acid sequences and whose carboxy termini consist of a foreign polypeptide. Antigenic epitopes of a subject polypeptide can then be detected with antibodies as for example reacting nitrocellulose filters lifted from infected plates with anti polypeptide antibodies. Phage scored by this assay can then be isolated from the infected plate. Thus the presence of gene homologs can be detected and cloned from other species and alternate isoforms including splicing variants can be detected and cloned.

The nucleic acid polypeptides and antibodies of the invention can be combined with other reagents and articles to form kits. Kits for diagnostic purposes typically comprise the nucleic acid polypeptides or antibodies in vials or other suitable vessels. Kits typically comprise other reagents for performing hybridization reactions polymerase chain reactions PCR or for reconstitution of lyophilized components such as aqueous media salts buffers and the like. Kits may also comprise reagents for sample processing such as detergents chaotropic salts and the like. Kits may also comprise immobilization means such as particles supports wells dipsticks and the like. Kits may also comprise labeling means such as dyes developing reagents radioisotopes fluorescent agents luminescent or chemiluminescent agents enzymes intercalating agents and the like. With the nucleic acid and amino acid sequence information provided herein individuals skilled in art can readily assemble kits to serve their particular purpose. Kits further can include instructions for use.

The nucleic acid sequence of the present invention may be used to detect or other species of acid sequence using bio chip technology. Bio chips containing arrays of nucleic acid sequence can also be used to measure expression of genes of or other species of . For example to diagnose a patient with a or other infection a sample from a human or animal can be used as a probe on a bio chip containing an array of nucleic acid sequence from the present invention. In addition a sample from a disease state can be compared to a sample from a non disease state which would help identify a gene that is up regulated or expressed in the disease state. This would provide valuable insight as to the mechanism by which the disease manifests. Changes in gene expression can also be used to identify critical pathways involved in drug transport or metabolism and may enable the identification of novel targets involved in virulence or host cell interactions involved in maintenance of an infection. Procedures using such techniques have been described by Brown et al. 1995270 467 470.

Bio chips can also be used to monitor the genetic changes of potential therapeutic compounds including deletions insertions or mismatches. Once the therapeutic is added to the patient changes to the genetic sequence can be evaluated for its efficacy. In addition the nucleic acid sequence of the present invention can be used to determine essential genes in cell cycling. As described in Iyer et al. 1999 283 83 87 genes essential in the cell cycle can be identified using bio chips. Furthermore the present invention provides nucleic acid sequence which can be used with bio chip technology to understand regulatory networks in bacteria measure the response to environmental signals or drugs as in drug screening and study virulence induction. Mons et al. 199816 45 48. Patents teaching this technology include U.S. Pat. Nos. 5 445 934 5 744 305 and 5 800 992.

By making available purified and recombinant polypeptides the present invention provides assays which can be used to screen for drugs which are either agonists or antagonists of the normal cellular function in this case of the subject polypeptides or of their role in intracellular signaling. Such inhibitors or potentiators may be useful as new therapeutic agents to combat infections in humans. A variety of assay formats will suffice and in light of the present inventions will be comprehended by the skilled artisan.

In many drug screening programs which test libraries of compounds and natural extracts high throughput assays are desirable in order to maximize the number of compounds surveyed in a given period of time. Assays which are performed in cell free systems such as may be derived with purified or semi purified proteins are often preferred as primary screens in that they can be generated to permit rapid development and relatively easy detection of an alteration in a molecular target which is mediated by a test compound. Moreover the effects of cellular toxicity and or bioavailability of the test compound can be generally ignored in the in vitro system the assay instead being focused primarily on the effect of the drug on the molecular target as may be manifest in an alteration of binding affinity with other proteins or change in enzymatic properties of the molecular target. Accordingly in an exemplary screening assay of the present invention the compound of interest is contacted with an isolated and purified polypeptide.

Screening assays can be constructed in vitro with a purified polypeptide or fragment thereof such as an polypeptide having enzymatic activity such that the activity of the polypeptide produces a detectable reaction product. The efficacy of the compound can be assessed by generating dose response curves from data obtained using various concentrations of the test compound. Moreover a control assay can also be performed to provide a baseline for comparison. Suitable products include those with distinctive absorption fluorescence or chemi luminescence properties for example because detection may be easily automated. A variety of synthetic or naturally occurring compounds can be tested in the assay to identify those which inhibit or potentiate the activity of the polypeptide. Some of these active compounds may directly or with chemical alterations to promote membrane permeability or solubility also inhibit or potentiate the same activity e.g. enzymatic activity in whole live cells.

Overexpression assays are based on the premise that overproduction of a protein would lead to a higher level of resistance to compounds that selectively interfere with the function of that protein. Overexpression assays may be used to identify compounds that interfere with the function of virtually any type of protein including without limitation enzymes receptors DNA or RNA binding proteins or any proteins that are directly or indirectly involved in regulating cell growth.

Typically two bacterial strains are constructed. One contains a single copy of the gene of interest and a second contains several copies of the same gene. Identification of useful inhibitory compounds of this type of assay is based on a comparison of the activity of a test compound in inhibiting growth and or viability of the two strains. The method involves constructing a nucleic acid vector that directs high level expression of a particular target nucleic acid. The vectors are then transformed into host cells in single or multiple copies to produce strains that express low to moderate and high levels of protein encoding by the target sequence strain A and B respectively . Nucleic acid comprising sequences encoding the target gene can of course be directly integrated into the host cell.

Large numbers of compounds or crude substances which may contain active compounds are screened for their effect on the growth of the two strains. Agents which interfere with an unrelated target equally inhibit the growth of both strains. Agents which interfere with the function of the target at high concentration should inhibit the growth of both strains. It should be possible however to titrate out the inhibitory effect of the compound in the overexpressing strain. That is if the compound is affecting the particular target that is being tested it should be possible to inhibit the growth of strain A at a concentration of the compound that allows strain B to grow.

Alternatively a bacterial strain is constructed that contains the gene of interest under the control of an inducible promoter. Identification of useful inhibitory agents using this type of assay is based on a comparison of the activity of a test compound in inhibiting growth and or viability of this strain under both inducing and non inducing conditions. The method involves constructing a nucleic acid vector that directs high level expression of a particular target nucleic acid. The vector is then transformed into host cells that are grown under both non inducing and inducing conditions conditions A and B respectively .

Large numbers of compounds or crude substances which may contain active compounds are screened for their effect on growth under these two conditions. Agents that interfere with the function of the target should inhibit growth under both conditions. It should be possible however to titrate out the inhibitory effect of the compound in the overexpressing strain. That is if the compound is affecting the particular target that is being tested it should be possible to inhibit growth under condition A at a concentration that allows the strain to grow under condition B.

Many of the targets according to the invention have functions that have not yet been identified. Ligand binding assays are useful to identify inhibitor compounds that interfere with the function of a particular target even when that function is unknown. These assays are designed to detect binding of test compounds to particular targets. The detection may involve direct measurement of binding. Alternatively indirect indications of binding may involve stabilization of protein structure or disruption of a biological function. Non limiting examples of useful ligand binding assays are detailed below.

A useful method for the detection and isolation of binding proteins is the Biomolecular Interaction Assay BLAcore system developed by Pharmacia Biosensor and described in the manufacturer s protocol LKB Pharmacia Sweden . The BIAcore system uses an affinity purified anti GST antibody to immobilize GST fusion proteins onto a sensor chip. The sensor utilizes surface plasmon resonance which is an optical phenomenon that detects changes in refractive indices. In accordance with the practice of the invention a protein of interest is coated onto a chip and test compounds are passed over the chip. Binding is detected by a change in the refractive index surface plasmon resonance .

A different type of ligand binding assay involves scintillation proximity assays SPA described in U.S. Pat. No. 4 568 649 .

Another type of ligand binding assay also undergoing development is based on the fact that proteins containing mitochondrial targeting signals are imported into isolated mitochondria in vitro Hurt et al. 19854 2061 2068 Eilers and Schatz 1986 322 228 231 . In a mitochondrial import assay expression vectors are constructed in which nucleic acids encoding particular target proteins are inserted downstream of sequences encoding mitochondrial import signals. The chimeric proteins are synthesized and tested for their ability to be imported into isolated mitochondria in the absence and presence of test compounds. A test compound that binds to the target protein should inhibit its uptake into isolated mitochondria in vitro.

Another ligand binding assay is the yeast two hybrid system Fields and Song 1989340 245 246 . The yeast two hybrid system takes advantage of the properties of the GAL4 protein of the yeast . The GAL4 protein is a transcriptional activator required for the expression of genes encoding enzymes of galactose utilization. This protein consists of two separable and functionally essential domains an N terminal domain which binds to specific DNA sequences UAS and a C terminal domain containing acidic regions which is necessary to activate transcription. The native GAL4 protein containing both domains is a potent activator of transcription when yeast are grown on galactose media. The N terminal domain binds to DNA in a sequence specific manner but is unable to activate transcription. The C terminal domain contains the activating regions but cannot activate transcription because it fails to be localized to UAS. In the two hybrid system a system of two hybrid proteins containing parts of GAL4 1 a GAL4 DNA binding domain fused to a protein X and 2 a GAL4 activation region fused to a protein Y . If X and Y can form a protein protein complex and reconstitute proximity of the GAL4 domains transcription of a gene regulated by UASoccurs. Creation of two hybrid proteins each containing one of the interacting proteins X and Y allows the activation region of UASto be brought to its normal site of action.

The binding assay described in Fodor et al. 1991251 767 773 which involves testing the binding affinity of test compounds for a plurality of defined polymers synthesized on a solid substrate may also be useful.

Compounds which bind to the polypeptides of the invention are potentially useful as antibacterial agents for use in therapeutic compositions.

Pharmaceutical formulations suitable for antibacterial therapy comprise the antibacterial agent in conjunction with one or more biologically acceptable carriers. Suitable biologically acceptable carriers include but are not limited to phosphate buffered saline saline deionized water or the like. Preferred biologically acceptable carriers are physiologically or pharmaceutically acceptable carriers.

The antibacterial compositions include an antibacterial effective amount of active agent. Antibacterial effective amounts are those quantities of the antibacterial agents of the present invention that afford prophylactic protection against bacterial infections or which result in amelioration or cure of an existing bacterial infection. This antibacterial effective amount will depend upon the agent the location and nature of the infection and the particular host. The amount can be determined by experimentation known in the art such as by establishing a matrix of dosages and frequencies and comparing a group of experimental units or subjects to each point in the matrix.

The antibacterial active agents or compositions can be formed into dosage unit forms such as for example creams ointments lotions powders liquids tablets capsules suppositories sprays aerosols or the like. If the antibacterial composition is formulated into a dosage unit form the dosage unit form may contain an antibacterial effective amount of active agent. Alternatively the dosage unit form may include less than such an amount if multiple dosage unit forms or multiple dosages are to be used to administer a total dosage of the active agent. Dosage unit forms can include in addition one or more excipient s diluent s disintegrant s lubricant s plasticizer s colorant s dosage vehicle s absorption enhancer s stabilizer s bactericide s or the like.

For general information concerning formulations see e.g. Gilman et al. eds. 19908th ed. Pergamon Press and 17th ed. 1990 Mack Publishing Co. Easton Pa. Avis et al. eds. 1993 Dekker New York Lieberman et al eds. 1990 Dekker New York.

The antibacterial agents and compositions of the present invention are useful for preventing or treating infections. Infection prevention methods incorporate a prophylactically effective amount of an antibacterial agent or composition. A prophylactically effective amount is an amount effective to prevent infection and will depend upon the specific bacterial strain the agent and the host. These amounts can be determined experimentally by methods known in the art and as described above.

The antibacterial agents and compositions can be administered topically or systemically. Topical application is typically achieved by administration of creams ointments lotions or sprays as described above. Systemic administration includes both oral and parental routes. Parental routes include without limitation subcutaneous intramuscular intraperitoneal intravenous transdermal inhalation and intranasal administration.

Following isolation whole genomic DNA was nebulized Bodenteich et al. Automated DNA Sequencing and Analysis J. C. Venter ed. Academic Press 1994 to a median size of 2000 bp. After nebulization the DNA was concentrated and separated on a standard 1 agarose gel. Several fractions corresponding to approximate sizes 1000 1500 bp 1500 2000 bp 2000 2500 bp 2500 3000 bp were excised from the gel and purified by the GeneClean procedure Bio101 Inc. .

The purified DNA fragments were then blunt ended using T4 DNA polymerase. The healed DNA was then ligated to unique BstXI linker adapters 5 GTCTTCACCACGGGG SEQ ID NO 5323 and 5 GTGGTGAAGAC SEQ ID NO 5324 in 100 1000 fold molar excess . These linkers are complimentary to the BstXI cut pMPX vectors while the overhang is not self complimentary. Therefore the linkers will not concatemerize nor will the cut vector religate itself easily. The linker adopted inserts were separated from the unincorporated linkers on a 1 agarose gel and purified using GeneClean. The linker adopted inserts were then ligated to each of 20 pMPX vectors to construct a series of shotgun subclone libraries. Blunt ended vector was used for cloning into the PUC19 vector. The vectors contain an out of frame lacZ gene at the cloning site which becomes in frame in the event that an adapter dimer is cloned allowing these to be avoided by their blue color.

All subsequent steps were based either on the multiplex DNA sequencing protocols outlined in Church G. M. and Kieffer Higgins S. Science 240 185 188 1988 or by ABI377 automated DNA sequencing methods. Only major modifications to the protocols are highlighted. Briefly each of the 20 vectors was then transformed into DH5a competent cells Gibco BRL DH5a transformation protocol . The libraries were assessed by plating onto antibiotic plates containing ampicillin methicillin and IPTG Xgal. The plates were incubated overnight at 37 C. Successful transformants were then used for plating of clones and pooling into the multiplex pools. The clones were picked and pooled into 40 ml growth medium cultures. The cultures were grown overnight at 37 C. DNA was purified using the Qiagen Midi prep kits and Tip 100 columns Qiagen Inc. . In this manner 100 mg of DNA was obtained per pool.

These purified DNA samples were then sequenced either using the multiplex DNA sequencing based on chemical degradation methods Church G. M. and Kieffer Higgins S. Science 240 185 188 1988 or by Sequithrem Epicenter Technologies dideoxy sequencing protocols or by ABI dye terminator chemistry. For the multiplex portion the sequencing reactions were electrophoresed and transferred onto nylon membranes by direct transfer electrophoresis from 40 cm gels Richterich P. and Church G. M. Methods in Enzymology 218 187 222 1993 . The DNA was covalently bound to the membranes by exposure to ultraviolet light and hybridized with labeled oligonucleotides complimentary to tag sequences on the vectors Church supra . The membranes were washed to rinse off non specifically bound probe and exposed to X ray film to visualize individual sequence ladders. After autoradiography the hybridized probe was removed by incubation at 65 C. and the hybridization cycle repeated with another tag sequence until the membrane had been probed 41 times. Thus each gel produced a large number of films each containing new sequencing information. Whenever a new blot was processed it was initially probed for an internal standard sequence added to each of the pools. Digital images of the films were generated using a laser scanning densitometer Molecular Dynamics Sunnyvale Calif. . The digitized images were processed on computer workstations VaxStation 4000 s using the program REPLICA Church et al. Automated DNA Sequencing and Analysis J. C. Venter ed. Academic Press 1994 . Image processing included lane straightening contrast adjustment to smooth out intensity differences and resolution enhancement by iterative gaussian deconvolution. The sequences were then converted to an SCF format so that processing and assembly could proceed on UNIX machines. The ABI dye terminator sequence reads were run on ABI377 machines and the data was directly transferred to UNIX machines following lane tracking of the gels. All multiplex and ABI reads were assembled using PHRAP P. Green Abstracts of DOE Human Genome Program Contractor Grantee Workshop V January 1996 p. 157 with default parameters and not using quality scores. The initial assembly was done at 7fold coverage and yielded 511 contigs. Short read length fragments of 200 bp or less found on the ends of contigs facing in the appropriate direction were used to extend off the end of the contigs. These reads were then resequenced with primers using ABI technology to give sequences with a read length of 500 or more bases. This allowed end extensions to be performed without ordering new primers. In addition missing mates sequences from clones that only gave one strand reads were identified and sequenced with ABI technology to allow the identification of additional overlapping contigs.

End sequencing of randomly picked genomic lambda was also performed. Sequencing on a both sides was done for all lambda sequences. The lambda library backbone helped to verify the integrity of the assembly and allowed closure of some of the physical gaps.

To identify polypeptides the complete genomic sequence of were analyzed essentially as follows First all possible stop to stop open reading frames ORFs greater than 180 nucleotides in all six reading frames were translated into amino acid sequences. Second the identified ORFs were analyzed for homology to known archeabacter prokaryotic and eukaryotic protein sequences. Third the predicted coding regions of the sequences and start codons were evaluated with the programs GENEMARK Borodovsky and Mclninch 1993 Comp. Chem. 17 123 and Glimmer Fraser et al 1997 .

Expression and purification of the polypeptides of the invention can be performed essentially as outlined below.

To facilitate the cloning expression and purification of membrane and secreted proteins from a gene expression system such as the pET System Novagen for cloning and expression of recombinant proteins in is selected. Also a DNA sequence encoding a peptide tag the His Tag is fused to the 3 end of DNA sequences of interest in order to facilitate purification of the recombinant protein products. The 3 end is selected for fusion in order to avoid alteration of any 5 terminal signal sequence.

Nucleic acids chosen for example from the nucleic acids set forth in SEQ ID NO 1 SEQ ID NO 2661 for cloning from the 14453 strain of are prepared for amplification cloning by polymerase chain reaction PCR . Synthetic oligonucleotide primers specific for the 5 and 3 ends of open reading frames ORFs are designed and purchased from GibcoBRL Life Technologies Gaithersburg Md. USA . All forward primers specific for the 5 end of the sequence are designed to include an NcoI cloning site at the extreme 5 terminus. These primers are designed to permit initiation of protein translation at a methionine residue followed by a valine residue and the coding sequence for the remainder of the native DNA sequence. All reverse primers specific for the 3 end of any ORF include a EcoRI site at the extreme 5 terminus to permit cloning of each sequence into the reading frame of the pET 28b. The pET 28b vector provides sequence encoding an additional 20 carboxy terminal amino acids including six histidine residues at the extreme C terminus which comprise the His Tag.

Genomic DNA prepared from strain 14453 of is used as the source of template DNA for PCR amplification reactions Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 . To amplify a DNA sequence containing an ORF genomic DNA 50 nanograms is introduced into a reaction vial containing 2 mM MgCl 1 micromolar synthetic oligonucleotide primers forward and reverse primers complementary to and flanking a defined ORF 0.2 mM of each deoxynucleotide triphosphate dATP dGTP dCTP dTTP and 2.5 units of heat stable DNA polymerase Amplitaq Roche Molecular Systems Inc. Branchburg N.J. USA in a final volume of 100 microliters.

Upon completion of thermal cycling reactions each sample of amplified DNA is washed and purified using the Qiaquick Spin PCR purification kit Qiagen Gaithersburg Md. USA . All amplified DNA samples are subjected to digestion with the restriction endonucleases e.g. NcoI and EcoRI New England BioLabs Beverly Mass. USA Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 . DNA samples are then subjected to electrophoresis on 1.0 NuSeive FMC BioProducts Rockland Me. USA agarose gels. DNA is visualized by exposure to ethidium bromide and long wave uv irradiation. DNA contained in slices isolated from the agarose gel is purified using the Bio 101 GeneClean Kit protocol Bio 101 Vista Calif. USA .

The pET 28b vector is prepared for cloning by digestion with endonucleases e.g. NcoI and EcoRI Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 . The pET 28a vector which encodes a His Tag that can be fused to the 5 end of an inserted gene is prepared by digestion with appropriate restriction endonucleases.

Following digestion DNA inserts are cloned Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 into the previously digested pET 28b expression vector. Products of the ligation reaction are then used to transform the BL21 strain of Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 as described below.

Competent bacteria strain BL21 or strain BL21 DE3 are transformed with recombinant pET expression plasmids carrying the cloned sequences according to standard methods Current Protocols in Molecular John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 . Briefly 1 microliter of ligation reaction is mixed with 50 microliters of electrocompetent cells and subjected to a high voltage pulse after which samples are incubated in 0.45 milliliters SOC medium 0.5 yeast extract 2.0 tryptone 10 mM NaCl 2.5 mM KCl 10 mM MgCl2 10 mM MgSO4 and 20 mM glucose at 37 C. with shaking for 1 hour. Samples are then spread on LB agar plates containing 25 microgram ml kanamycin sulfate for growth overnight. Transformed colonies of BL21 are then picked and analyzed to evaluate cloned inserts as described below.

Individual BL21 clones transformed with recombinant pET 28b ORFs are analyzed by PCR amplification of the cloned inserts using the same forward and reverse primers specific for each sequence that were used in the original PCR amplification cloning reactions. Successful amplification verifies the integration of the sequences in the expression vector Current Protocols in Molecular Biology John Wiley and Sons Inc. F. Ausubel et al. eds. 1994 .

Individual clones of recombinant pET 28b vectors carrying properly cloned ORFs are picked and incubated in 5 mls of LB broth plus 25 microgram ml kanamycin sulfate overnight. The following day plasmid DNA is isolated and purified using the Qiagen plasmid purification protocol Qiagen Inc. Chatsworth Calif. USA .

The pET vector can be propagated in any K 12 strain e.g. HMS174 HB101 JM109 DH5 etc. for the purpose of cloning or plasmid preparation. Hosts for expression include strains containing a chromosomal copy of the gene for T7 RNA polymerase. These hosts are lysogens of bacteriophage DE3 a lambda derivative that carries the lacI gene the lacUV5 promoter and the gene for T7 RNA polymerase. T7 RNA polymerase is induced by addition of isopropyl B D thiogalactoside IPTG and the T7 RNA polymerase transcribes any target plasmid such as pET 28b carrying its gene of interest. Strains used include BL21 DE3 Studier F. W. Rosenberg A. H. Dunn J. J. and Dubendorff J. W. 1990 Meth. Enzymol. 185 60 89 .

To express recombinant sequences 50 nanograms of plasmid DNA isolated as described above is used to transform competent BL21 DE3 bacteria as described above provided by Novagen as part of the pET expression system kit . The lacZ gene beta galactosidase is expressed in the pET System as described for the recombinant constructions. Transformed cells are cultured in SOC medium for 1 hour and the culture is then plated on LB plates containing 25 micrograms ml kanamycin sulfate. The following day bacterial colonies are pooled and grown in LB medium containing kanamycin sulfate 25 micrograms ml to an optical density at 600 nM of 0.5 to 1.0 O.D. units at which point I millimolar IPTG was added to the culture for 3 hours to induce gene expression of the recombinant DNA constructions.

After induction of gene expression with IPTG bacteria are pelleted by centrifugation in a Sorvall RC 3B centrifuge at 3500 g for 15 minutes at 4 C. Pellets are resuspended in 50 milliliters of cold 10 mM Tris HCl pH 8.0 0.1 M NaCl and 0.1 mM EDTA STE buffer . Cells are then centrifuged at 2000 g for 20 min at 4 C. Wet pellets are weighed and frozen at 80 C. until ready for protein purification.

A variety of methodologies known in the art can be utilized to purify the isolated proteins. Current Protocols in Protein Science John Wiley and Sons Inc. J. E. Coligan et al. eds. 1995 . For example the frozen cells may be thawed resupended in buffer and ruptured by several passages through a small volume microfluidizer Model M 110S Microfluidics International Corporation Newton Mass. . The resultant homogenate may be centrifuged to yield a clear supernatant crude extract and following filtration the crude extract may be fractionated over columns. Fractions may be monitored by absorbance at ODnm. and peak fractions may analyzed by SDS PAGE.

The concentrations of purified protein preparations may be quantified spectrophotometrically using absorbance coefficients calculated from amino acid content Perkins S. J. 1986 Eur. J. Biochem. 157 169 180 . Protein concentrations are also measured by the method of Bradford M. M. 1976 Anal. Biochem. 72 248 254 and Lowry O. H. Rosebrough N. Farr A. L. Randall R. J. 1951 J. Biol. Chem. 193 pages 265 275 using bovine serum albumin as a standard.

SDS polyacrylamide gels of various concentrations may be purchased from BioRad Hercules Calif. USA and stained with Coomassie blue. Molecular weight markers may include rabbit skeletal muscle myosin 200 kDa gal actosidase 116 kDa rabbit muscle phosphorylase B 97.4 kDa bovine serum albumin 66.2 kDa ovalbumin 45 kDa bovine carbonic anhydrase 31 kDa soybean trypsin inhibitor 21.5 kDa egg white lysozyme 14.4 kDa and bovine aprotinin 6.5 kDa .

While this invention has been particularly shown and described with references to preferred embodiments thereof it will be understood by those skilled in the art that various changes in form and details may be made therein without departing from the scope of the invention encompassed by the appended claims.

